---
layout: post
title:  "BBa_J04450"
author: Reporter
categories: [collection 1 - 2021,cds, reporter, chromoprotein, classic, reporter, pret, function, reporter, color, function, reporter, pigment] 
image: assets/images/6.jpg
---


  * [main page](http://parts.igem.org/Part:BBa J04450)
  * [design](http://parts.igem.org/Part:BBa J04450:Design)
  * [experience](http://parts.igem.org/Part:BBa J04450:Experience)
  * [information](http://parts.igem.org/cgi/partsdb/part_info.cgi?part_name=BBa J04450)
  *   * 

[Released HQ 2013](http://parts.igem.org/Help:Part_Status_Box)

[Sample In stock](http://parts.igem.org/Help:Part_Status_Box)

![](http://parts.igem.org/images/stars/1star.png)1 Registry Star

[69 Uses](http://parts.igem.org/partsdb/uses.cgi?part=BBa_J04450)

[9 Twins](http://parts.igem.org/partsdb/twin_info.cgi?part=BBa_J04450)

[ Get This Part](http://parts.igem.org/partsdb/get_part.cgi?part=BBa_J04450)

![](http://parts.igem.org/images/partbypart/icon_reporter.png)

Reporter

# Part:BBa_J04450

Designed by: Tamar Odle   Group: iGEM2005   (2005-06-09)

  
RFP Coding Device  
  

  

Assembly Compatibility:

  * 10

COMPATIBLE WITH RFC[10]

  * 12

COMPATIBLE WITH RFC[12]

  * 21

COMPATIBLE WITH RFC[21]

  * 23

COMPATIBLE WITH RFC[23]

  * 25

INCOMPATIBLE WITH RFC[25]

Illegal AgeI site found at 781  
Illegal AgeI site found at 893  

  * 1000

COMPATIBLE WITH RFC[1000]

The colonies are clearly red in color under natural light after about 18
hours. Smaller colonies are visibly red under UV. The RFP part does not
contain a degradation tag and the RBS is strong.

  * LacI sensitive
  * CAP sensitive

  
  

# Improvement by JNFLS2020

## Overview:

  
BBa_J04450 is a RFP generator driven by Plac promoter, which is regulated by
LacI and induced by IPGT. Plac is one of the most common promoter in life
science research field. It is mainly composed of Lac operon containing LacO
site. LacI repressor, encoded by LacI gene, can bind to LacO site, inhibiting
the binding of RNA polymerase to the promoter. So the expression of downstream
genes is blocked. Serving as inducer, IPTG can bind with LacI repressor,
making the latter’s conformation changes, so LacI is detached from LacO site,
which enables the transcription of downstream genes. BBa_J04450 is a RFP
generator driven by Plac promoter, however there is no LacI gene in it.
Although the E. coli could express some LacI, it is not enough for inhibition
RFP expression. So this RFP generator has some leakage expression, like the
designer stated that some RFP express even without IPTG inducer presence. This
year, we constructed a new RFP generator (BBa_K3605010) also driven by Plac
promoter. It contains LacI gene, which can lower significantly the RFP leakage
expression. Both LacI and RFP are linked to the downstream of Plac, which is
regulated by both LacI repressor and IPTG inducer. Using RFP as a reporter,
its fluorescence intensity appears a lower leakage expression. This new RFP
generator could be self-regulated because LacI protein can inhibit its self-
expression, so that no excessive LacI expression which is considered as waste
of resources. We detected the response of this generator to different
concentration of IPTG, indicating that it could be inhibited by LacI, and
induced well by IPTG.  

## Results:

  
Overlapping PCR was used to clone the LacI and RFP genes. Then the PCR product
was inserted to plasmid pSB1C3 to create the new part BBa_K3605010. The
identification result is showed in Fig.1.  

[![K3605010-1.jpg](/wiki/images/0/0e/K3605010-1.jpg)](/File:K3605010-1.jpg)

Fig.1. The result of BBa_K3605010 construction. M: Marker; 1: Digestion of
BBa_K3605010; 2: Plasmid of BBa_K3605010.  
  
We compared the inducing effect of IPTG on the two RFP generators, using
different concentration of IPTG. We set 5 groups: 4 experimental groups of the
old RFP generator (BBa_Jo4450) and the new RFP generator (BBa_K3605010) with
or without IPTG, one negative control without RFP expression. At 0h, all
groups’ OD600 approximately reaches to 0.6, then certain concentration of IPTG
was added to the culture medium, incubated cells at 37℃ for 20h. Measure the
fluorometric value at 610 nm and OD600 value for each group every 2h, using an
automatic microplate reader. The results are showed as follows (Fig.2-Fig.4).  

[![K3605010-2.jpg](/wiki/images/d/df/K3605010-2.jpg)](/File:K3605010-2.jpg)

Fig.2. The inducing effect of IPTG on the two RFP generators. Relative
fluorescent intensity is fluorescence per OD600 standardized with fluorescence
per OD600 value of each test group at time = 0, IPTG=0. The figure indicated
that both BBa_J04450 and BBa_K3605010 expressed RFP induced by IPTG, and
different concentration of IPTG had same inducing trend. Then we detected the
RFP expression with IPTG presence or absence, using BBa_J04450 and
BBa_K3605010, respectively. The results showed that BBa_J04450 expressed a
mountain of RFP whether IPTG is present or not, which means that BBa_J04450
had a high leakage expression and lower sensitivity to the IPTG induction.
However, BBa_K3605010 expressed a high level RFP only with IPTG presence. The
leakage expression is very low, and it is sensitive to the IPTG induction.  
  

[![K3605010-3.jpg](/wiki/images/7/70/K3605010-3.jpg)](/File:K3605010-3.jpg)

Fig.3. The comparison of RFP expression using BBa_J04450 with and without IPTG
induction. Relative fluorescent intensity is fluorescence per OD600
standardized with fluorescence per OD600 value of each test group at time = 0,
IPTG=0. This figure indicated that the RFP generator (BBa_J04450) had a high
leakage expression, and low sensitivity to the IPTG induction.  
  

[![K3605010-4.jpg](/wiki/images/5/53/K3605010-4.jpg)](/File:K3605010-4.jpg)

Fig.4. The comparison of RFP expression using BBa_K3605010 with and without
IPTG induction. Relative fluorescent intensity is fluorescence per OD600
standardized with fluorescence per OD600 value of each test group at time = 0,
IPTG=0. This figure indicated that the new RFP generator (BBa_K3605010) is
very sensitive to the IPTG induction, and the RFP leakage expression is very
low.  
  

## Protocol:

  1. Transform the plasmids into BL21 strain.
  2. Pick up a single colony by a sterile tip from each of the plates for all the experimental and control groups. And put the colony into 5ml M9 medium with 100 µg/ml ampicillin. Incubate at 37℃ in a shaker for 20h.
  3. Detect OD600value of the culture medium with spectrophotometer, and dilute the culture medium to OD600= 0.6.
  4. Add 100 µl bacteria culture medium into a sterile 96-well plate. IPTG is added to final concentrations of 1, 2, 5, 10mM. M9 medium is the blank control. The colony without RFP expression is the negative control.
  5. Incubate at 37℃, and measure the fluorometric value at 610 nm and OD600value for each well every 2h, using an automatic microplate reader.
  6. The experiment should be repeated at least 3 times.

  
  
  
  
This part is commonly used, but can fail if the system contains LacI or CAP
protein.  
(\--[Meagan](/User:Meagan "User:Meagan") 15:39, 23 July 2009 (UTC))

[Team TU_Munich 2012](http://2012.igem.org/Team:TU_Munich) improved this part
by making it compatible to RFC10 and RFC25 (see:
[BBa_K801100](http://parts.igem.org/wiki/index.php/Part:BBa_K801100))  
(\--[VolkerMorath](/User:VolkerMorath "User:VolkerMorath") 15:02, 21 October
2012 (UTC))

[Team NRP-UEA 2013](http://2013.igem.org/Team:NRP-UEA-Norwich) improved this
part by adding a NdeI restriction site before the RFP gene. (see:
[BBa_K1041000](http://parts.igem.org/wiki/index.php/Part:BBa_K1041000))  
(\--[holusac](/wiki/index.php?title=User:Holusac&action=edit&redlink=1
"User:Holusac \(page does not exist\)") 20:46, 14 August 2013 (UTC))

[Team Warwick 2015](http://2015.igem.org/Team:Warwick) improved this part by
analysing the effect of copy number on gene expression.  
(\--[Lcarroll](/wiki/index.php?title=User:Lcarroll&action=edit&redlink=1
"User:Lcarroll \(page does not exist\)") 20:48, 25 September 2015 (UTC))

[Team Leiden 2016](http://2016.igem.org/Team:Leiden) contributed to the
characterisation of this part by showing equal functionality in simulated
microgravity (0g) as in the normal gravity of the Earth.  
(\--[Valentijn](/wiki/index.php?title=User:Valentijn&action=edit&redlink=1
"User:Valentijn \(page does not exist\)") 19:38, 19 October 2016 (UTC))

[Team UChicago 2017](http://2017.igem.org/Team:UChicago) contributed to this
part by improving/changing the documented sequence through mutagenesis to
create blunt-end restriction sites for cloning not within the prefix/suffix
region (created BBa_K2428000).  
(\--[pzulueta97](/wiki/index.php?title=User:Pzulueta97&action=edit&redlink=1
"User:Pzulueta97 \(page does not exist\)") 21:14, 25 October 2017 (UTC) )

[Team Grenoble-Alpes 2017](http://2017.igem.org/Team:Grenoble-Alpes)
contributed to the characterisation of this part by testing the time of
apparition of fluorescence, in presence of IPTG or not (because the promoter
leaks), as well as they contributed to the improvement of this part by using
its fluorescence as a detection signal to be able to detect Vibrio Cholerae.  
(\--[NoreenLouis](/wiki/index.php?title=User:NoreenLouis&action=edit&redlink=1
"User:NoreenLouis \(page does not exist\)") 20:47, 26 October 2017 (UTC) )

[Team Kingsborough NY 2017](http://2017.igem.org/Team:Kingsborough_NY)
contributed to the characterization of this part by showing decreased
fluorescence when expressed either in a higher salt media - such as LB with 3%
sodium chloride - or E. coli that lacks tmRNA, the principal component of the
cell's ribosome rescue system. View the data on the experience page or [visit
our Wiki](http://2017.igem.org/Team:Kingsborough_NY/RFP)  
(\--[djcamenares](/wiki/index.php?title=User:Djcamenares&action=edit&redlink=1
"User:Djcamenares \(page does not exist\)") 17:56, 27 October 2017 (UTC) )

[Team iTesla SoundBio 2017](http://2017.igem.org/Team:iTesla-SoundBio)
contributed to the characterization of this part by analyzing the rate of
false positives when using the coloring of transformed colonies as a red/white
screen for determining experimental success.  
(\--[gladish](/wiki/index.php?title=User:Gladish&action=edit&redlink=1
"User:Gladish \(page does not exist\)") 01:26, 28 October 2017 (UTC) )

[Team Grenoble-Alpes 2018](http://2018.igem.org/Team:Grenoble-Alpes)
contributed to the characterisation of this part by testing the delay before
apparition of fluorescence directly after transformation and the intensity of
the leak, in three different E. Coli strains.  
(\--[perrine](/wiki/index.php?title=User:Perrine&action=edit&redlink=1
"User:Perrine \(page does not exist\)") 15:06, 9 October 2018 (UTC) )

[Team H14Z1_Hangzhou 2018](http://2018.igem.org/Team:H14Z1_Hangzhou)
contributed to the characterisation of this part by testing the fluorescence
idensity in different condition(e.g.temperature,medium volume, IPTG
concentration) ,in two different E. Coli strains.  
(\--[ericxu](/wiki/index.php?title=User:Ericxu&action=edit&redlink=1
"User:Ericxu \(page does not exist\)") 09:32, 17 October 2018 (UTC) )

[Team SKLMT-China 2018](http://2018.igem.org/Team:SKLMT-China) improved this
part by replacing the induced promoter lacI with the strong constitutive pf-5
promoter PampC.(see:
[BBa_K2569029](http://parts.igem.org/wiki/index.php/Part:BBa_K2569029))  
(\--[DDY](/wiki/index.php?title=User:DDY&action=edit&redlink=1 "User:DDY
\(page does not exist\)") 16:47, 17 October 2018 (UTC) )

[Team Grenoble-Alpes 2019](https://2019.igem.org/Team:Grenoble-
Alpes/Contribution) contributed to the characterisation of this part by
testing the effect of cAMP on the promoter’s leakages and protein expression
in Escherichia coli.  
(\--[pinerol](/wiki/index.php?title=User:Pinerol&action=edit&redlink=1
"User:Pinerol \(page does not exist\)") 19:50, 27 September 2019 (UTC) )

[Team Lund 2019](https://2019.igem.org/Team:Lund) contributed to the
charazterization of this part by testing the effect of different media
composition in different _E. coli_ strains on RFP expression.  
(\--[
ParselTMatts](/wiki/index.php?title=User:ParselTMatts&action=edit&redlink=1
"User:ParselTMatts \(page does not exist\)") 18:40, 19 October 2019 (UTC) )

[Team BUCT-China 2019](https://2019.igem.org/Team:BUCT-China) Contribute to
the characterization of this part by introducing it in Trans10 to observe its
growth and draw its growth curve.  
(\--[
buct_china_ljh](/wiki/index.php?title=User:Buct_china_ljh&action=edit&redlink=1
"User:Buct china ljh \(page does not exist\)") 12:18, 20 October 2019 (UTC) )

[Team Aalto-Helsinki 2019](https://2019.igem.org/Team:Aalto-Helsinki)
contributed to the charazterization of this part by testing the effects of
different chloramphenicol concentrations on RFP expression levels.  
(\--[ Toivanen](/wiki/index.php?title=User:Toivanen&action=edit&redlink=1
"User:Toivanen \(page does not exist\)") 15:20, 21. October 2019 (UTC) )

[Team NAU_CHINA 2019](https://2019.igem.org/Team:NAU-CHINA) contributed to the
characterization of this part by testing the effect of glucose on the protein
expression in E. coli.  
(\--[ NiuDan](/wiki/index.php?title=User:NiuDan&action=edit&redlink=1
"User:NiuDan \(page does not exist\)") 17:41, 21. October 2019 (UTC) )

[Team Botchan_Lab_Tokyo 2019](https://2019.igem.org/Team:Botchan_Lab_Tokyo)
contributed to characterization of this part by testing radiation tolerance of
_E.coli_ express the RFP.  
(\--[ youma](/wiki/index.php?title=User:Youma&action=edit&redlink=1
"User:Youma \(page does not exist\)") 21:20, 21. October 2019 (UTC) )

## Pictures

  * [![](/wiki/images/thumb/7/7e/J04450_-_lightbox.jpg/120px-J04450_-_lightbox.jpg)](/File:J04450_-_lightbox.jpg)

BBa_J04450 visualized under non-UV lightbox

  * [![](/wiki/images/thumb/e/e9/J04450_-_UV_254nm.jpg/120px-J04450_-_UV_254nm.jpg)](/File:J04450_-_UV_254nm.jpg)

BBa_J04450 visualized under 254nm wavelength UV lightbox

  * [![](/wiki/images/thumb/2/2d/J04500-colonies.jpg/120px-J04500-colonies.jpg)](/File:J04500-colonies.jpg)

BBa_J04450 Colonies

  * [![](/wiki/images/thumb/3/3b/T--USP-Brazil--Registry--RFP-pagglomerans.jpeg/120px-T--USP-Brazil--Registry--RFP-pagglomerans.jpeg)](/File:T--USP-Brazil--Registry--RFP-pagglomerans.jpeg)

_Pantoea agglomerans_ colonies in chloramphenicol, containing BBa_J04450 in
the pSB1C3 backbone

  * [![](/wiki/images/thumb/a/a1/T--Tec-Chihuahua--RFP.jpeg/120px-T--Tec-Chihuahua--RFP.jpeg)](/File:T--Tec-Chihuahua--RFP.jpeg)

_E.Coli_ SHuffle colonies transformed with BBa_J04450

## Alma 2020 Improvement: Successful Replication of reverted pSB1C3 in Vibrio
Natriegens

We have been unable to transform J04450 into Vibrio natriegens, despite being
able to get other ColE1 and pUC19 origin based plasmids in this species. We
have discovered that pSB1C3 contains a point mutation in the origin of
replication that could explain this effect. This part is a reversion of this
mutation, making pSB1C3 into a true pUC19 origin plasmid. We proceeded to test
this by transforming by electroporation into Vibrio natriegens, as well as
measuring plasmid copy number.

  * [![T--Alma--mhplate1.png](/wiki/images/thumb/4/4f/T--Alma--mhplate1.png/120px-T--Alma--mhplate1.png.jpeg)](/File:T--Alma--mhplate1.png)

  * [![T--Alma--mhplate2.png](/wiki/images/thumb/5/57/T--Alma--mhplate2.png/120px-T--Alma--mhplate2.png.jpeg)](/File:T--Alma--mhplate2.png)

  * [![T--Alma--mhplate3.png](/wiki/images/thumb/c/c1/T--Alma--mhplate3.png/120px-T--Alma--mhplate3.png.jpeg)](/File:T--Alma--mhplate3.png)

The results above are representative of several experiments. As can be seen,
J04450 cannot be transformed into Vibrio, while our mutant was able to
successfully produce colonies. In these results, “-“ is the negative control
plate streaked with pSB1C3 containing J04450 (“RFP”). This is the most
commonly used backbone for iGEM BioBricks. The plate contains no growth. “+”
contains pGGA without the plasmid that expresses RFP. This contains the pUC19
origin, which has expression in vibrio. However, pGGA is not compatible with
iGEM BioBricks. “Mr” (BBa_K3445002) is the mutated backbone (BBa_K3445000) to
both contain the gene needed to express RFP and allow for compatibility to
show expression in vibrio. We observed in these experiments a red color in
Vibrio, indicating that the bacteria can express the RFP gene.

We also measured the plasmid copy number of our mutant, both in E. coli and in
Vibrio. We did so by performing qPCR on DNA isolated from equal numbers of
these bacteria using either genome targeting primers ( **dnaE** ) or plasmid
targeting primers ( **oriQ** ), as done in previous studies. The copy number
is thus measured as 2 raised to the difference between the oriQ Ct value and
the dnaE Ct value.

  * [![T--Alma--plot1.png](/wiki/images/thumb/5/59/T--Alma--plot1.png/120px-T--Alma--plot1.png.jpeg)](/File:T--Alma--plot1.png)

  * [![T--Alma--plot2.png](/wiki/images/thumb/6/60/T--Alma--plot2.png/120px-T--Alma--plot2.png.jpeg)](/File:T--Alma--plot2.png)

  * [![T--Alma--plot3.png](/wiki/images/thumb/8/8a/T--Alma--plot3.png/120px-T--Alma--plot3.png.jpeg)](/File:T--Alma--plot3.png)

In _E. Coli_ , the copies of plasmid per genome for J04450 with a pSB1C3
backbone was 6.92 copies as measured by qPCR. For the mutated backbone
containing J04450 (“K3445002”), the plasmid per genome measured was 12.21,
25.11, and 10.85 copies. This averages to roughly 16.05 copies of plasmid per
genome shown in e. Coli for K3445002. In vibrio, the reported values for
plasmid per genome for pGGA was found to be 306 copies. The found values for
copies of plasmid per genome for K3445002 in vibrio could not be determined in
practice due to there not being a reported Ct value for K3445002 with the DNA
E primer. However, an estimated value was determined by using the Ct value for
pGGA with the DNA E primer and the Ct value for K3445002 with the OriQ primer.
The estimated value for K3445002 in vibrio was determined to be 6.73 copies of
plasmid per genome. These values indicate that K3445002 is effective in both
_E. Coli_ and _Vibrio natriegens_ , and there may even be a higher plasmid
copy number in E. Coli with K3445002 in relation to the copy number for
J04450. The dissociation curve was analyzed to verify that each reaction in
the qPCR only produced one product. Taken together, these results show that we
have improved the backbone of all parts (and J04450 specifically), resulting
in a plasmid that is still capable of replication in _E. coli_ , but now also
available for use in _Vibrio natriegens_.

## Team Amazonas_Brazil 2019:The RFP sinalization system in E. coli Nissle
1917

### Characterization

Escherichia coli Nissle 1917 (EcN) is widely used as a probiotic since it’s
discovery in the second half of the 19th century by Alfred Nissle1 and
although many iGEM teams had worked with this bacteria, we couldn’t find any
specific characterization about the manipulation of EcN. To improve the
quality of our work and provide data for other teams or researchers that might
want to work with this bacteria in their laboratory, we characterized EcN
growth curve and it’s behavior when transformed by electroporation with the
iGEM vector pSB1C3 with GFP and RFP.  
E. coli Nissle 1917 strains were grown overnight in Lysogeny Broth (LB)
containing ampicilin (100 µg/mL) at 37°C and 200 rpm. Cultures were diluted in
fresh LB until achieve 0,1 OD with the corresponding antibiotic and
transferred to a 96-well plate (50 µL/well). Samples were always made in
triplicates and a blank of LB. During 8h the absorbance at OD600 and
fluorescence (excitation 584 nm and emission 607 nm) were measured with
intervals of 1 hour.  
![](https://2019.igem.org/wiki/images/f/fb/T--Amazonas-Brazil--expressao-
rfp.png)  
Fig. 1. Fluorescence values of E. coli Nissle 1917 curve(w/v)  

## Team NAU_CHINA 2019:The inhibitory effect of different concentrations of
glucose in LB culture

### Characterization

We used liquid M9 medium several times in our experiments, and the M9 medium
contained a small amount of glucose (as a carbon source for bacteria). We used
it to cultivate DH5α containing J04450 standard part; however, J04450 part
contains a CAP-sensitive promoter, and glucose can inhibit its expression.  
Since we were interested in the fact that glucose inhibits the expression of
J04450, we explored the inhibitory effect of different concentrations of
glucose on J04450 in LB culture. Add 50 ul of chloramphenicol to 5 bottles of
LB medium and add IPTG to reach the concentration of 1%. Add 40% glucose to
the concentration of 0,0.5%,2%,4%(w/v). Inoculate 1 ml of bacterial fluid and
incubate at 37℃,150 rpm for every 0,8 h,16h,24h,28h,32h,36h. The phenotype was
observed at 40h, and the change in fluorescence value was quantified by a
microplate reader. ![](https://2019.igem.org/wiki/images/d/da/T--NAU-CHINA--
Characterization-2.jpg)  
Fig. 1. OD600 values of bacteria at different glucose concentrations(w/v)  
![](https://2019.igem.org/wiki/images/7/78/T--NAU-CHINA--
Characterization-3.jpg)  
Fig. 2. Fluorescence intensity of bacteria at different glucose
concentrations(w/v)  
![](https://2019.igem.org/wiki/images/1/19/T--NAU-CHINA--
Characterization-4.jpg)  
Fig. 3. Fluorescence intensity/OD value of bacteria at different glucose
concentrations(w/v)

### Analysis

We measured the OD value and fluorescence value of the sample by the
microplate reader, and finally obtained the OD-time curve, fluorescence value-
time curve, and fluorescence value/OD-time curve.  
The data shows that the OD600 value of the cells decreases in the order of
glucose concentration of 2.0%, 1.0%, 0.5%, 0, 4.0%, indicating that the proper
amount of glucose can promote the growth of the cells, but the excess glucose
has an inhibitory effect. The fluorescence intensity/OD value shows that
different concentrations of glucose inhibited the expression of J04450, and
the inhibition effect was the most significant at 4.0%.

### Reference

[1]Mustea I , Muresian T . Crabtree effect in some bacterial cultures[J].
CANCER, 1967, 20(9):1499-1501.

## Team ITB_Indonesia 2017: Red color dynamics of cloned _Escherichia coli_
strains in LB broth

![Three strains expression on
mRFP](http://parts.igem.org/wiki/images/5/53/Barenganstrain.JPG) _Background_  
  
In [Team ITB_Indonesia 2017](http://2017.igem.org/Team:ITB_Indonesia)
characterization, we found in normal growth/incubation condition (37 oC, LB
agar) that BBa_J04450-transformed _Escherichia coli_ BL21 colony appear to
need longer incubation time (>18 hours) until it clearly shows red color under
natural light.  
  
We then investigate whether this phenomenon is influenced by the strain, and
we try if there are _lac_ repressor in the system that can be released by
inducing the culture with IPTG, hence increasing the expression of mRFP.  
  
_Experimental Design_  
  
We used three different strains of transformed _E. coli_ (BL21, DH5alpha, and
Top10) for this study. They were incubated in LB broth, 37 oC, and sampled
every 4 hours for 2 days to determine the red color absorbance at 588 nm. The
amount of IPTG added for respective treatment is 500 µM.  
  
_Result and Findings_  

  * There are no significant differences of mRFP expression in different strains of _E. coli_ (BL21, DH5alpha, Top10)
  * There are no significant effects of mRFP increased expression after IPTG induction.
  * The red color absorbance under 588 nm wavelength is recorded around 2.5-3 OD units.
  * The broth become red in color under natural light around 16-20 hours of incubation time.

  
![](http://parts.igem.org/wiki/images/7/72/Top10woIPTG.JPG)
![](http://parts.igem.org/wiki/images/b/bd/DH5woIPTG.jpeg)
![](http://parts.igem.org/wiki/images/b/b7/BL21wolIPTG.jpeg) ==Pictures==  
![](

                    http://2017.igem.org/wiki/images/0/06/Foto_mRFP1.jpg) _**Figure 2:** ~18 hours old mRFP-transformed E. coli strains (Right: control)_ ![](
                    http://2017.igem.org/wiki/images/2/2c/Foto_mRFP2.jpg) _**Figure 3:** ~30 hours old mRFP-transformed E. coli strains (Left: control). Notice the color turn to bright purple._

## Team INSA-UPS France 2017 : usage in _Vibrio harveyi_ strain engineered by
conjugation

In the context of the iGEM INSA-UPS France project 2017, BBa_J04450 was tested
in the _Vibrio harveyi_ background. To the best of our knowledge, RFP has
never been used in this strain. BBa_J04450 biobrick was cloned in a broad host
range plasmid (pBBR1MCS-4) and conjugated into _V. harveyi_. The protocol of
triparental mating can be found [here](http://2017.igem.org/Team:INSA-
UPS_France/Experiments#protocols). Its expression has been studied by
fluoresence microscopy in _Vibrio harveyi_ .

[![](/wiki/images/1/19/T--INSA-UPS_France--RFPHarveyi.png)](/File:T--INSA-
UPS_France--RFPHarveyi.png)

[](/File:T--INSA-UPS_France--RFPHarveyi.png "Enlarge")

**Monitoring of the production of RFP by _V. harveyi_ strain microscopy **
_Vibrio harveyi_ JMH626 conjugated with pBBR1MCS-4-RFP (panels C & D) or with
pBBR1MCS-4 (control ; panels A & B) was analyzed by optic microscopy (panel A
& C) or fluorescence microscopy (panels B & D).

The microscopy results demonstrated the fonctional production of RFP in
_Vibrio harveyi_ , and hence, the functionality of part BBa_J04450 in this
background.

## IIT Madras 2016's Characterization

[![Specific growth rates \(ln\) for
Devices](/wiki/images/thumb/d/d4/Sp_growth.png/400px-
Sp_growth.png)](/File:Sp_growth.png "Specific growth rates \(ln\) for
Devices")

[![Strength of RFP when used along with various other
devices](/wiki/images/thumb/6/6a/Iitm_rfp_perplasmid.png/400px-
Iitm_rfp_perplasmid.png)](/File:Iitm_rfp_perplasmid.png "Strength of RFP when
used along with various other devices")

## Experimentation

This BioBrick was used along with various GFP producing devices to understand
the role of RBS and Promoter parts in giving rise to intrinsic noise in _E.
coli_ DH5alpha. Expression data for GFP and RFP proteins were obtained using
flow cytometry (BD FACS Aria III) at 3hr, 6hr, 9hr and 12hr stage of growth
along with cells expressing only GFP, only RFP and none. Cumulative intrinsic
and extrinsic noise were measured using modified [Elowitz
formula](http://2016.igem.org/Team:IIT-Madras/Model#Noise_in_Devices%7C).
OD600 values for specific growth rate estimation were obtained using
Spectrophotometer over an interval of an hour for 12 hours. Given specific
growth rates are in it's logarithmic values. This BioBrick can be used to
characterize noise and strength of complex devices by cloning this device with
given device, which produces a different reporter protein. In graphs, we have
[R11-B32](http://parts.igem.org/Part:BBa_K1956014),
[R11-B34](http://parts.igem.org/Part:BBa_K1956015),
[J14-B3](http://parts.igem.org/Part:BBa_K1956017),
[J17-B34](http://parts.igem.org/Part:BBa_K1956018),
[R11-B30](http://parts.igem.org/Part:BBa_K1956022) and
[R11-B31](http://parts.igem.org/Part:BBa_K1956023) in pSB1A2 plasmid backbone.

[Team Groningen 2010](http://2010.igem.org/Team:Groningen) reports the usage
of this part as a cloning tool. When ligating any part, or part assembly, into
any standard backbone that contains this part, the non-restricted and single-
restricted backbones that self-circularize will produce red colonies on rich
media plates (we use TY). These undesired transformants can than be avoided in
the screening for the correct construct. With this method, the backbone
desired for a new construct does not need to be purified from agarose gel to
decrease the amount of undesired tranformants caused by ligation of the
original part present in the backbone. The amount of incorrect transformants
depends, of course, on the ratio of backbone (mixed with J04450) vs. BioBrick
insert, the size of the BioBrick insert, and whether the insert is an assembly
of two BioBricks. The images below show two ligations with different
efficiencies.

[![](/wiki/images/thumb/0/0f/Co-inef-gn.jpg/200px-Co-inef-gn.jpg)](/File:Co-
inef-gn.jpg)

[](/File:Co-inef-gn.jpg "Enlarge")

An inefficient assembly ligation of two BioBricks into the pSB1C3 backbone
producing many red colonies.

[![](/wiki/images/thumb/2/28/Co-ef-gn.jpg/200px-Co-ef-gn.jpg)](/File:Co-ef-
gn.jpg)

[](/File:Co-ef-gn.jpg "Enlarge")

A more efficient, single BioBrick ligation into the pSB1C3 backbone.

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

## Usage in Chromobacterium Violaceum

[Team Tec-Monterrey 2016](http://2016.igem.org/Team:Tec-Monterrey)
characterized the output of the part BBa_J04450 in a novel chassis,
Chromobacterium Violaceum, as it produces a native purple pigment Violacein,
we were curious whether RFP would be useful as a reporter gene. Furthermore,
we characterized its expression under lac promoter. We did the transformation
of C. Violaceum by a method that has not been reported yet, we made C.
Violaceum competent cells with the protocol that is in our wiki, we concluded
that the best O.D. for the heat shock transformation is 0.5 since it showed
clearly better results than 0.4 or 0.6, we will continue to work in the
transformation efficiency.

[Image:![610px-T--Tec-Monterrey--
2016_CalciumResults_Cviolaceum.png](http://2016.igem.org/wiki/images/thumb/e/ee/T
--Tec-Monterrey--2016_CalciumResults_Cviolaceum.png/610px-T--Tec-Monterrey--
2016_CalciumResults_Cviolaceum.png)]

## H14Z1_Hangzhous 2018's Characterization

[Team H14Z1_Hangzhou 2018](http://2018.igem.org/Team:H14Z1_Hangzhou)  
  

_Purpose_  
  

Previous experiments had shown that BBa_J04450-transformed _E. coli_ BL21(DE3)
would turn red color under natural light after 18 hours due to leaking
expression. This part is useful for characterizing the expression of different
proteins using the lacI sensitive promoter. For better application of this
promoter in controlling protein expression, the effects of different hosts
under various conditions were investigated systematically. The target hosts
include _E. coli_ BL21(DE3) and _E. coli_ MC1061. The target culture
conditions were temperature, inducer concentration and medium volume in the
flask.  
  

_Experiment design_  
  

  * Exploring the effect of temperatures on mRFP expression  
1.Inoculate 5 ml LB with monocolony (BL21(DE3)/MC1061) from the plate. Grow
the cells overnight at oC，200 rpm.  
2.Dilute pre-culture 1:100 into 5ml LB (BL21(DE3)/MC1061) and cultivate at
different temperature (26, 30, 34, 37 oC), 200 rpm, respectively.  
3.Measure the fluorescence and OD600 every 4 hours.

  * Exploring the effect of medium volume on mRFP expression  
1.Inoculate 5 ml LB with monocolony (BL21/MC1061) from the plate.Grow the
cells overnight at 37oC, 200 rpm.  
2.Dilute pre-culture 1:100 into 30ml, 50ml, 70ml LB (BL21(DE3)/MC1061) and
cultivate at 37oC, 200 rpm.  
3.Measure the fluorescence and OD600 every 2 hours.

  * Exploring the effect of concentration of IPTG on mRFP expression  
1.Inoculate 5 ml LB with monocolony(BL21(DE3)/MC1061) from the plate. Grow the
cells overnight at 37oC, 200 rpm.  
2.Dilute pre-culture 1:100 into 5ml LB (BL21(DE3)/MC1061), cultivate at 37oC,
200 rpm. Protein expression are induced using 0.1, 0.3, 0.5, 0.8, 1.0 mM IPTG,
respectively.  
3.Measure the fluorescence and OD600 every 4 hours.

  

_Results_  
  

  * Exploring the effect of temperatures on mRFP expression

There are some differences of growth rate at different temperatures for both
_E. coli_ (BL21(DE3), MC1061) (Fig.1). However, the differences of the
fluorescence intensity at different temperature is much higher than those of
the growth rate at various temperatures by using two E. coli cells (Fig 2).
The fluorescence intensity followed same trend in two _E. coli_
(BL21(DE3)/MC1061).The results indicated that the leaking expression was lower
in low temperature (e.g.26 and 30oC) and higher in high temperature. Highest
leak appeared in 34 oC. This finding is different from our expectation, where
we expected that the highest leaking expression would be at 37 oC

[![](/wiki/images/thumb/8/8f/T--H14Z1_Hangzhou--mRFP_OD.jpeg/500px-T--
H14Z1_Hangzhou--mRFP_OD.jpeg)](/File:T--H14Z1_Hangzhou--mRFP_OD.jpeg)

[](/File:T--H14Z1_Hangzhou--mRFP_OD.jpeg "Enlarge")

**Figure.1 Effect of different temperature on cell growth**

[![](/wiki/images/thumb/f/f6/T--H14Z1_Hangzhou--mRFP_TEMP_FLU.jpeg/500px-T--
H14Z1_Hangzhou--mRFP_TEMP_FLU.jpeg)](/File:T--H14Z1_Hangzhou--
mRFP_TEMP_FLU.jpeg)

[](/File:T--H14Z1_Hangzhou--mRFP_TEMP_FLU.jpeg "Enlarge")

**Figure.2 Efforts of different temperature on mRFP expression**

  

  

  * Exploring the effect of effect of medium volume on mRFP expression

The mRFP expression of two strains showed the same trend in different volume
of medium in the flask. After 6 hours, the fluorescence intensity in 30 ml
medium-containing flask is significantly higher than others, indicating that
mRFP express fastest in this volume scale (Fig 3). This suggest that dissolved
oxygen is an important factor which will affect the protein expressin and cell
growth, since the lower medium volume in the same type flask will bring about
higher level of dissolved oxygen in the medium.

[![](/wiki/images/thumb/d/d8/T--H14Z1_Hangzhou--mRFP_Volume_Flu.jpeg/500px-T--
H14Z1_Hangzhou--mRFP_Volume_Flu.jpeg)](/File:T--H14Z1_Hangzhou--
mRFP_Volume_Flu.jpeg)

[](/File:T--H14Z1_Hangzhou--mRFP_Volume_Flu.jpeg "Enlarge")

**Figure.3 Effects of medium volume in the flask on mRFP expression**

  

  * Exploring the effect of concentration of IPTG on mRFP expression 

Our results showed that significant fluorescence differences were not observed
by using different IPTG induction (Fig 3). However, it is interesting that the
fluorescence intensity of _E. coli_ BL21(DE3) after IPTG induction are higher
than those of _E. coli_ MC1061, not similar to the trend in our previous
experiment (Fig 2). It might be explained that the expression of mRFP can be
accelerated by IPTG inductin in BL21(DE3) with regard to other _E. coli_
hosts.

[![](/wiki/images/thumb/4/47/T--H14Z1_Hangzhou--mRFP_IPTG_Flu.jpeg/500px-T--
H14Z1_Hangzhou--mRFP_IPTG_Flu.jpeg)](/File:T--H14Z1_Hangzhou--
mRFP_IPTG_Flu.jpeg)

[](/File:T--H14Z1_Hangzhou--mRFP_IPTG_Flu.jpeg "Enlarge")

**Figure.4 Efforts of different IPTG concentration on mRFP expression**

  

_Pictures_  
  

[![](/wiki/images/thumb/5/57/T--H14Z1_Hangzhou--mRFP_VOLUME-PIC.jpeg/500px-T--
H14Z1_Hangzhou--mRFP_VOLUME-PIC.jpeg)](/File:T--H14Z1_Hangzhou--mRFP_VOLUME-
PIC.jpeg)

[](/File:T--H14Z1_Hangzhou--mRFP_VOLUME-PIC.jpeg "Enlarge")

**Figure.5 Efforts of different temperature on mRFP expression**

  

[![](/wiki/images/thumb/a/ab/T--H14Z1_Hangzhou--mRFP_Temp-PIC.jpeg/400px-T--
H14Z1_Hangzhou--mRFP_Temp-PIC.jpeg)](/File:T--H14Z1_Hangzhou--mRFP_Temp-
PIC.jpeg)

[](/File:T--H14Z1_Hangzhou--mRFP_Temp-PIC.jpeg "Enlarge")

**Figure.6 Effects of medium volume in the flask on mRFP expression**

  

[![](/wiki/images/thumb/a/a1/T--H14Z1_Hangzhou--mRFP_IPTGE-PIC.jpeg/500px-T--
H14Z1_Hangzhou--mRFP_IPTGE-PIC.jpeg)](/File:T--H14Z1_Hangzhou--mRFP_IPTGE-
PIC.jpeg)

[](/File:T--H14Z1_Hangzhou--mRFP_IPTGE-PIC.jpeg "Enlarge")

**Figure.7 Efforts of different IPTG concentration on mRFP expression**

  

  

## SCAU-China 2018's improvement

[Team:SCAU-China 2018](http://parts.igem.org/Part:BBa_K2559005#)

The [BBa_K2559005](http://parts.igem.org/Part:BBa_K2559005#) is a amended eGFP
coding part improved from BBa_l714891.

### Usage and Biology

The part BBa_K2559005 has a sequence improvement on the basic part submitted
by iGEM07_Peking ([BBa_l714891](http://parts.igem.org/Part:BBa_l714891#))
which encodes the SDY_eGFP. However, we found out a 16 bp nucleotides
redundancy in the eGFP starting coding region in BBa_I714891, after checking
the sequence of BBa_I714891 from NCBI. Therefore, we decided to delete the
redundant 16 bp nucleotides in BBa_I714891 to amend the length of eGFP coding
sequence. The amended eGFP coding biobrick is the BBa_K2559005. To test the
function of BBa_K2559005, we designed a new E.coli expression vector
containing our new part termed as
[BBa_K2559003](http://parts.igem.org/Part:BBa_K2559003#)under a strong E.coli
endogenous promoter (PrplJ). Therefore, the amended eGFP in BBa_K2559005 was
driven by PrplJ promoter, and expressed in DH10B. In addition, we also applied
the BBa_K2559005 in the promoter intensity analysis of our other two new
parts, the [BBa_K2559004](http://parts.igem.org/Part:BBa_K2559004#) and
[BBa_K2559011](http://parts.igem.org/Part:BBa_K2559011#) which are relatively
weaker E.coli endogenous promoters (PdapA and PcaiF) (Figure 1).

[![Scau-china-2018-11.png](/wiki/images/c/ce/Scau-
china-2018-11.png)](/File:Scau-china-2018-11.png)

[](/File:Scau-china-2018-11.png "Enlarge")

  

[![](/wiki/images/f/f7/Scau-china-2018-12.png)](/File:Scau-china-2018-12.png)

[](/File:Scau-china-2018-12.png "Enlarge")

Figure 1: Fluorescent intensity of amended eGFP driven by by PrplJ, PdapA,
PcaiF promoter.

  

We summarized that our improvedpart, the amended eGFP coding biobrick
BBa_K2559005 worked well in DH10B. We also hoped that our improvement on the
BBa_I714891 can help their future applications by other groups in the future.
However, it is difficult for us to perform additional experiments with
BBa_K2559005 and BBa_I714891 due to the unavailable BBa_I714891.

To expand the application of BBa_K2559005, we searched theBBa_J04450 stored in
registry and do another improvement in the BBa_J04450. The BBa_J04450 is a
strong RFP expression vector in E.coli. As the main page of BBa_J04450
mentioned, the E.coli colonies with BBa_J04450 were in red color under normal
light after about 18 hour culture on LB plate (Figure 2). We used the
BBa_K2559005 to replace the RFP region in BBa_J04450, the modified part is
[BBa_K2559009](http://parts.igem.org/Part:BBa_K2559009#). We transferred the
BBa_K2559009 to DH5α by heat-shock, and found that the fluorescence signal can
be observed under the UV (Figure 2).

[![](/wiki/images/thumb/b/b1/T-SCAU-China-Red_and_green_colonies.jpg/200px-T-
SCAU-China-Red_and_green_colonies.jpg)](/File:T-SCAU-China-
Red_and_green_colonies.jpg)

[](/File:T-SCAU-China-Red_and_green_colonies.jpg "Enlarge")

Figure 2 : Figure 2 Colonies with BBa_J04450 (red colony) and BBa_K2559009
(green colony) were visualized under UV lightbox

So, we confirm that our improved part BBa_K2559005 can work in different
E.coli expression system. We are also looking forward to more application of
the BBa_K2559009! </html>

  

## Grenoble-Alpes 2019 Characterization

  
**The goal was to characterize the effect of cAMP on the promoter’s leakages
and overall proteins expression**  
  
_**MAIN RESULTS**_  
[More informations here](https://2019.igem.org/Team:Grenoble-
Alpes/Contribution) or in experience page.  
_**DH5α**_ : standard strain with endogenous adenylate cyclase activity.  

_**BTH101**_ : strain without endogenous adenylate cyclase activity and so
without production of cAMP. _BTH101_ strain is streptomycin resistant.  

_**BTH101**_ \+ adenylate cyclase ( _BTH101_ -Zip) to generate cAMP in the
strain. To restore the adenylate cyclase activity we used two plasmids: one
containing _pUT18-LeucineZipper_ (pUT18-LZ) and the other containing
_pKT25-LeucineZipper_ (pKT25-LZ).  
The Leucine Zipper’s (LZ) will homodimerize and will bring T18 and T25 closer
which will restore the adenylate cyclase activity thus allowing the production
of cAMP by _BTH101_. Both T18 and T25 are under the control of an IPTG
inducible promoter (lactose promoter) to create an auto amplifier system.  
_BBa_J04450_ was cloned in _pKT25-LeucineZipper_ , and the bacteria was co-
transformed with both plasmid _T25-LeucineZipper + BBa_J04450_ and
_T18-LeucineZipper_.  
  
![800px-T--Grenoble-Alpes--
Transfo_charac.png.jpeg](http://parts.igem.org/wiki/images/thumb/1/17/T--
Grenoble-Alpes--Transfo_charac.png/800px-T--Grenoble-Alpes--
Transfo_charac.png.jpeg)  
  
It is apparent that without IPTG there is a visible difference between strains
that can produce cAMP ( _DH5α_ and _BTH101_ -Zip) when compared with the
_BTH101_ strain that do not produce any cAMP.  
The _DH5α red_ colonies reveal a substantial RFP expression as the result of
the leakage of the promoter.  
_BTH101_ has white colonies indicating that no visible RFP was produced. This
data suggest that the promoter does not leak in that condition.  
_BTH101_ -Zip presents same red colonies than the _DH5α_ strain.  
  
![800px-T--Grenoble-Alpes--
Charac_conclu.png](http://parts.igem.org/wiki/images/thumb/1/16/T--Grenoble-
Alpes--Charac_conclu.png/800px-T--Grenoble-Alpes--Charac_conclu.png)  
  
RFP expression by _BTH101_ with or without cAMP shows that without cAMP, the
lactose promoter has almost no leakage suggesting that the leakage of the
lactose promoter is due to cAMP in absence of IPTG. When the cAMP production
is restored the promoter is activated and the protein is expressed. It is also
interesting to note that with the _BT101_ -Zip there are very little protein
expression if no IPTG is added.  

## Tec-Chihuahua 2019 Characterization: RFP expression in _E.Coli_ SHuffle
strain

**Introduction**

_E. coli_ SHuffle T7 Express®️ form New England BioLabs®️ was chosen for its
ability to produce disulfide bonded proteins in its cytoplasm. However, no
information was found in the Registry about the usage of part BBa_R0010 in
SHuffle strains. In order to test this lacI regulated promoter in our chassis,
we transformed part BBa_J04450, which expresses RFP under this promoter, into
E. coli SHuffle T7 Express®️.Parts BBa_R0010 and BBa_J04450 were
characterized. RFP expression was induced in different temperature conditions
and IPTG concentration to determine the ideal conditions for the proper
functionality of the promoter. Protein expression was analysed through an SDS-
PAGE.

**Experimentation**

First, 3 μL of the 10 pg/μL vial of Part BBa_J04450 (RFP construct) from the
Competent Cell Kit were transformed into chemically competent _E. coli_
SHuffle T7 Express®️ cells. Transformed cells were grown on LB agar with
chloramphenicol, for BBa_J04450 was located in the pSB1C3 backbone. Small,
rounded, and red colonies grew on the surface of the agar.

  

[![T--Tec-Chihuahua--RFP.jpeg](/wiki/images/thumb/a/a1/T--Tec-Chihuahua--
RFP.jpeg/300px-T--Tec-Chihuahua--RFP.jpeg)](/File:T--Tec-Chihuahua--RFP.jpeg)
**Figure 1.** _E.Coli_ Shuffle colonies transformed with BBa_J04450.

  

Once the expression chassis was transformed, protein expression was induced.
To do so, two tubes of 10 mL of liquid culture of untransformed _E. coli_
SHuffle T7 Express®️ and six of SHuffle cells transformed with BBa_J04450 were
prepared. Once the tubes reached an OD600 of 0.4, they were incubated for 5
hours under the conditions of temperature and IPTG concentration shown in the
following table.  
  

  

**Table 1.** Conditions of induction for characterization of production of
RFP. Well | 37°C | Well | 30°C  
---|---|---|---  
Negative control |  Untransformed control |  Negative control |  Untransformed
control  
1  |  Uninduced control (without IPTG) |  4  |  Uninduced control (without
IPTG)  
2  | Transformed cells, 0.2 mM IPTG |  5  | Transformed cells, 0.2 mM IPTG  
3  | Transformed cells, 0.4 mM IPTG |  6  | Transformed cells, 0.4 mM IPTG  
  
  

Induction temperatures were chosen based on commonly used conditions (37°C)
and recommendations of the supplier (30°C). The uninduced control consists of
transformed cells without IPTG which meant to serve as an indicator of
leakiness in the promoter. IPTG concentrations were chosen based on conditions
worked previously when using a T7 promoter (0.2 mM) and recommendations of the
supplier (0.4 mM).  
After 5 hours of incubation at 225 rpm, cells were centrifuged at 5000 rpm and
4°C for 7 minutes. The pellet was resuspended in a lysis solution with
lysozyme and cells were lysed through multiple freeze/thaw cycles. The cell
lysates were centrifuged at 21,380 g to separate the soluble protein fraction
(supernatant) which was retrieved for its evaluation and quantification. 10 μL
of each sample with 50 μg of total protein were mixed with 10 μL of SDS
loading buffer and loaded into a 12% polyacrylamide gel. The gel was ran at 10
mA until the running front entered the separating gel; then, the current was
increased to 15 mA.

  
**Results**

The following image shows the results obtained from the SDS-PAGE.

  

[![T--Tec-Chihuahua--RFPGEL.jpeg](/wiki/images/thumb/b/b4/T--Tec-Chihuahua--
RFPGEL.jpeg/550px-T--Tec-Chihuahua--RFPGEL.jpeg)](/File:T--Tec-Chihuahua--
RFPGEL.jpeg) **Figure 2.** SDS-PAGE (12%) of E. coli SHuffle T7 Express®️
transformed with BBa_J04450. (Negative control: _E.coli_ Shuffle untransformed
cells at 37°C, 1.RFP transformed cells without induction, 2.RFP transformed
cells with 0.2mM of IPTG at 37°C, 3.RFP transformed cells with 0.4mM of IPTG
at 37°C, Negative control: _E.coli_ Shuffle untransformed cells at 30°C, 4.RFP
transformed cells without induction, 5.RFP transformed cells with 0.2mM of
IPTG at 30°C, 6.RFP transformed cells with 0.4mM of IPTG at 30°C)

  

Two prominent bands can be appreciated in the last two columns, corresponding
to induction conditions of 0.2 and 0.4 mM IPTG at 30°C, close to the 25 kDa
band of the Precision Plus ProteinTM Dual Xtra Prestained Protein Standards.
It was concluded that they correspond to RFP, a 26 kDa protein.

The band corresponding to RFP can be seen in each column with exception of the
untransformed cells controls (negative controls). The uninduced controls
(column 1 for 37°C and column 4 for 30°C) without IPTG were expected not to
show bands if the promoter lacked leakiness; nevertheless, a faint band can be
seen for both controls. Bands corresponding to 0.2 and 0.4 mM IPTG for both
temperatures (columns 2 and 3 for 37°C and columns 5 and 6 for 30°C) show
stronger bands compared to the uninduced control. Thus, the promoter is
slightly leaky, but the addition of proper amounts of IPTG increases protein
production. When comparing temperatures, incubation at 30°C clearly yields a
higher amount of RFP, but yields similar amounts of expression between both
concentrations.

  
**Conclusion**

We have demonstrated the functionality of promoter BBa_R0010 in E. coli
SHuffle T7 Express®️ as well as ideal conditions for protein production by
characterizing part BBa_J04450, an RFP construct. E. coli SHuffle T7 Express®️
produces higher amounts of protein at 30°C, and concentrations of 0.2 and 0.4
mM IPTG are enough to induce expression when using this lacI regulated
promoter.

## **Team Lund 2019 charecterization**

### Aim

The aim of this characterization was to investigate the correlation between
the different strains and how the varying amounts of salts and sugars in the
media stimulate cell growth and protein expression. RFP was incorporated in
the construct to compare the protein expression in various conditions. Three
different E.coli strains were tested in three different complex media. The
strains that were tested: E. coli TG1 E. coli BL21-DE3 E. coli Nissle 1917
(EcN) While the prepared media were: LB Media TB Media Enriched Complex Media
The recipe for the three types of media can be found in our
[protocols](https://2019.igem.org/Team:Lund/Experiments).

### Method

The plasmid from the registry was successfully transformed using heat shock to
each of the mentioned strains. Three 1 L Erlenmeyer flasks containing 250mL of
each media were prepared (9 flasks in total). Cultivation was performed at a
temperature of 37 °C and 175 rotations per minute (rpm) , during which the
growth was monitored spectrophotometrically. The protein expression could be
detected optically from the color formation as well as with the use of Agilent
Technologies Cary Eclipse Fluorescence Spectrophotometer at excitation
wavelength of 532 nm and emission wavelength of 605 nm. For this experiment, a
wavelength of 660 nm was used to measure optical cell density (OD) following
the suggestion from iGEM headquarters.

The calibration curves for Rhodamine B (dye equivalent of RFP, obtained from
Thermo Fisher Scientific - D1841) and for microbeads (equivalent of cells,
provided by iGEM headquarters) were used to create a conversion factor for
auxiliary fluorescence unit and Abs 660 nm. Two excitation wavelengths have
been used 532 nm (recommended for RFP) and 570 nm (recommended for Rhodamine
B). The protocol used to make the standard curves was the iGEM protocol “iGEM
2019 Plate Reader Fluorescence Calibration” for RFP.

### Results

**Standard curve**

The emissions for these two excitation wavelengths can be seen in Figure 1
below.

[![532-570-RFP.png](/wiki/images/1/17/532-570-RFP.png)](/File:532-570-RFP.png)

_Figure.1 The emission spectra for various concentrations of Rhodamine B (0 to
50μM): a) excitation at 532nm; b) excitation at 570nm._

One can notice that at excitation wavelength of 532 nm, a clear emission peak
can be obtained approximately at 590 nm with the increase in signal intensity
for each Rhodamine B concentration. While at excitation wavelength of 570 nm
there are no clear peaks but the signal increases with the concentration of
the fluorescent dye. Therefore, several standard curves have been plotted. The
532 nm/605 nm (ex./em.) was the one that was used for RFP, the 605 nm was a
result of fluorescence shift due to presence of RFP in the cells. The 532
nm/590 nm was the setting to check the theoretical emission Rhodamine B at 590
nm using the same excitation as RFP. The last set of wavelengths 570 nm/590 nm
was recommended by the manufacturer of Rhodamine B. The results are presented
in Figure 2 below.

[![RhodamineB-standard.png](/wiki/images/7/7f/RhodamineB-
standard.png)](/File:RhodamineB-standard.png)

_Figure 2. Rhodamine B standard curves showing mean fluorescence of four
replicates of different concentrations of Rhodamine B (0 µM to 50 µM): a)
excitation: 532nm, emission: 605nm; b) excitation: 532nm, emission: 590;c)
excitation: 570, emission: 590_

It can be clearly seen that in all cases there was an oversaturation of
detector at concentrations of 12.5 μM and above. Therefore, it was decided
that these concentrations should not be used for creation of standard curve
and have been excluded. In case of 532/605, the noise is disturbing the
measurement at lower concentrations but both 532/590 and 570/590 do not seem
to have the same issues.

[![RhodmanieB-fluorescence.png](/wiki/images/d/dc/RhodmanieB-
fluorescence.png)](/File:RhodmanieB-fluorescence.png)

_Figure 3. Rhodamine B standard curves showing mean fluorescence of four
replicates of different concentrations of Rhodamine B (0 µM to 6.25 µM – the
higher values were excluded due to oversaturation):a) excitation: 532 nm,
emission: 605 nm; b) excitation: 532 nm, emission: 590 nm;c) excitation: 570
nm, emission: 590 nm_

It can be seen that all the curves show similar linearity. The data from the
curves was used to determine a unit scaling factor (μM Rhodamine B/
fluorescence a.u.) that can be found in Table 1, below.

Table 1. The mean μM of Rhodamine B per fluorescence auxiliary unit (a.u.)

[![TableRFP.png](/wiki/images/b/bf/TableRFP.png)](/File:TableRFP.png)

Since all the standard curves have similar linearity but the scaling factors
are significantly different, it was decided to choose the manufacturers
recommendation and use the date from 570nm excitation and 590nm emission, that
is 0.23 μM Rhodamine B/a.u.. It was hence used to convert the fluorescence
auxiliary units in the characterisation experiment.

The calibration curve for the particles can be seen in Figure 4. below.

[![ParticlesStandard.png](/wiki/images/8/87/ParticlesStandard.png)](/File:ParticlesStandard.png)

_Figure 4. The calibration curve for absorbance of various concentrations
silica particles at 660 nm._

The calculated unit conversion factor was 1,23E+09 particles/Abs660 and it was
used to convert units in characterization experiments.

The RFP/Rhodamine B Fluorescence Standard Spreadsheets found here for all
excitation/emission can be found here:

[532/605](https://2019.igem.org/File:T--Lund--
FluorescenceStandard532-605.xlsx) [532/590](https://2019.igem.org/File:T--Lund
--FluorescenceStandard532-590.xlsx) [570/590](https://2019.igem.org/File:T--
Lund--FluorescenceStandard570-590.xlsx)

  

**Characterization results**

Figure 5a below shows the growth of EcN in different media in a fraction of
time. Enriched Complex Media appears to be the most suitable for the growth of
EcN. The lag phase is shorter compared to the other two media, while the
logarithmic phase is longer. In TB Media the lag phase is extended, and the
growth is not as high as before but still exhibits a clear logarithmic phase.
In sharp contrast to that, the lag phase in LB Media is almost double and the
exponential phase is quite brief. These results are expected since the media
most enriched with salts and sugars are the media with the highest growth
rate. The same applies to all strains (Figure 5b and Figure 5c).

[![RFP-OD.png](/wiki/images/3/30/RFP-OD.png)](/File:RFP-OD.png)

_Figure 5. The growth of various E. coli strains expressing RFP in different
media: a) EcN in LB (blue), TB (orange), EM (grey). b) E. coli BL21-DE3 in LB
(blue), TB (orange), EM (grey) c) E. coli TG1 in LB (blue), TB (orange), EM
(grey). The OD was converted to equivalent particle count/100μL._

Figure 6 below shows the expression of RFP in different media. In EcN (Figure
6a), the expression of RFP for TB and Enriched Media are comparable, whereas
the RFP production in LB is significantly lower. For E. coli BL21-DE3 (Figure
6b), however, the expression levels in TB and LB Media are similar and in
Enriched media the RFP production was considerably higher. Finally, in E. coli
TG1 (Figure 6c) contrary to the other two strains the Enriched Media gives the
lowest RFP levels. The expression in LB media is only slightly higher than in
Enriched media. The protein expression in TB Media, however, is notably higher
than the other two media, and in comparison to the other strains in the same
media. In all three strains, the production of the protein was initiated in LB
Media but it could not reach as high values as in the more fortified media,
which was expected.

[![StrainsRB.png](/wiki/images/b/bf/StrainsRB.png)](/File:StrainsRB.png)

_Figure 6. The expression of RFP in various E. coli strains in various
cultivation media: a) EcN in LB (blue), TB (orange), EM (grey). b) E. coli
BL21-DE3 EcN in LB (blue), TB (orange), EM (grey) c) E. coli TG1 EcN in LB
(blue), TB (orange), EM (grey). The OD was converted to equivalent particle
count/100μL._

Figure 7 below shows the expression of RFP (converted to Rhodamine B
equivalents) per OD (converted to particle/100uL). When it comes to growth,
all strains prefered Enriched media. However, it can be noted that each strain
has a different preference regarding the RFP expression. EcN produces the most
RFP per cell in TB media, BL21 in LB media while in TG1 the expression is
favored in both LB and TB media.

[![RelativeF.png](/wiki/images/9/92/RelativeF.png)](/File:RelativeF.png)

_Figure 7. The production of RFP per particle count (OD equivalent) for each
strain in various media at the end of cultivation: LB (blue), TB (orange), EM
(grey)._

Figure 8 displays growth of the three strains of E.coli during the
characterization of RFP. Overall, the darkest red color was from cells grown
in TB media. This was where the protein expression was higher. The other two
media, complex Enriched media and LB media both gave a quite pale pink color
and low protein expression, with one exception of the BL21-DE3 cells which
gave a dark red color and high protein expression. For BL21-DE3 cells, the
complex enriched media showed the darkest color. For TG1 cells the TB media
gave the darkest color. For EcN cells the TB media also gave the darkest
color. These correspond to the results presented in Figure 7.

[![FlasksFluorecscence.png](/wiki/images/f/f3/FlasksFluorecscence.png)](/File:FlasksFluorecscence.png)

_Figure 8. Growth of three strains of E.coli cells transformed with RFP
plasmid BBa_J04450. The cells were grown in different media. The three flasks
on the left were grown in TB media, and the strains were from left to right;
BL21-DE3, TG1, EcN. The three flasks in the middle were grown in complex
Enriched media, and the strains were from left to right; BL21-DE3, TG1, EcN.
The three flasks on the right were grown in LB media, and the strains were
from left to right; BL21-DE3, TG1, EcN._

### Usage and Biology

**Using cAMP depleted strains and T18-leucineZipper/T25-leucineZipper under
the control of a lactose promoter is a good alternative to produce inducible
proteins without leakage**  

## **Team BUCT_China 2019**

### Experimental Design

1.The target gene was obtained by PCR cloning.  
2\. Digest the gene and plasmid pSB1C3 with enzymes Dpn,IEcoR,Pst I.  
3.Ligate the digested gene fragment and plasmid pSB1C3 with T4 enzyme.  
4.Transfer the ligation product to Trans 10 by thermal conversion.  
5.Plate the transformed strain and culture at 37℃ for 12h.  
6.Pick a single colony from the plate to make a bacterial solution.The cells
were cultured separately in 24 tubes, and every tube was measured the
absorbance in order for each 1 hour.  

### Results

The growth of the plate colonies was observed as shown in the Figure a. We
could see that an appropriate amount of colonies grew on the plate, indicating
that the target gene fragment was successfully cloned into Trans 10.  

[![T--BUCT-China--Kit1.jpg](/wiki/images/thumb/d/d2/T--BUCT-China--
Kit1.jpg/500px-T--BUCT-China--Kit1.jpg)](/File:T--BUCT-China--Kit1.jpg)

[](/File:T--BUCT-China--Kit1.jpg "Enlarge")

**Figure a：** ~12 hours old transformed Trans 10 strains.

  
  
We quantitatively described the growth curve of Trans10 single colony in
medium.(Figure b) According to the growth rate constant of the bacteria, we
could divide 24h into three different growth phases：0~2h is lag phase; 2~ 15h
is an exponential phase; 15~ 24 is a stationary phase.  

[![T--BUCT-China--Kit4.jpg](/wiki/images/6/6f/T--BUCT-China--
Kit4.jpg)](/File:T--BUCT-China--Kit4.jpg) **Figure b：** The growth curve of a
single colony from the plate.

  

## **Newcastle 2019 characterisation**

The Newcastle iGEM team aimed to develop a suite of biosensors using
fluorescence proteins as a reporter. Our goal was to investigated if we can
measure fluorescence level correctly when fluorescence proteins are combined.
We investigate whether mixing cultures of cells expressing different
fluorescent proteins would affect the fluorescence intensity of the individual
fluorescent proteins. The fluorescent proteins we chose were sfGFP
(BBa_K515105) and mCherry (BBa_J04450).

We obtained sfGFP and mCherry into pSB1AT3 from iGEM distribution kits. The
resuspended DNA was transformed into _E. coli_ DH5 alpha cells. Cells were
inoculated in LB media and grown to an optical density 600 (OD600) of 0.6.
Half of the culture was diluted to OD600 0.3 and the individual fluorescence
levels were measured at: sfGFP - excitation 480nm and emission 507 nm and
mCherry – excitation 580 nm, emission 610 nm. The sfGFP and mCherry cultures
at OD600 0.6 were mixed together in equal amounts and the fluorescence level
of the combined mixture was measured in a BioTek Synergy H1 Microplate Reader
with a gain of 100.

The sfGFP and mCherry cultures at OD600 0.6 were mixed together in equal
amounts and the fluorescence level of the combined mixture was measured.
Theoretically, the OD600 of each fluorescence protein in the mixture will be
0.3. By comparing the individual fluorescence level at OD600 of 0.3 to the
fluorescence level of the equivalent fluorescence protein minus the opposing
fluorescent protein, the effect of mixing fluorescent proteins can be
observed.

The emission spectra for sfGFP ranges from 469 nm to 628 nm and the emission
spectra for mCherry ranges from 551 nm to 800 nm. There is an overlap of 77 nm
in the emission spectra of the two fluorescent proteins. This small overlap
suggests that there will not be much effect on the fluorescence intensity of
the individual fluorescent proteins as the overlap will be at an emission with
a low percentage emission.

[![2019sfGFPandmCherryMixing.png](/wiki/images/thumb/b/bd/2019sfGFPandmCherryMixing.png/600px-2019sfGFPandmCherryMixing.png)](/File:2019sfGFPandmCherryMixing.png)

[](/File:2019sfGFPandmCherryMixing.png "Enlarge")

_Figure 1. A) Bar chart showing the mean fluorescence intensity of E. coli DH5
alpha cells expressing sfGFP individually at an OD 600 of 0.3 with standard
deviations compared to the mean fluorescence intensity when mixed with cells
expressing the fluorescent protein mCherry Excitation 480 nm, emission 507. B)
Bar chart showing the mean fluorescence intensity of E. coli DH5 alpha cells
expressing mCherry individually at an OD600 of 0.3 with standard deviations
compared to the mean fluorescence intensity when mixed with cells expressing
the fluorescent protein sfGFP. Excitation 580 nm, emission 610 nm. _

The results of mixing sfGFP and mCherry at sfGFP excitation and emission
(Figure 1A) showed that there was no significant difference in fluorescence
intensity of sfGFP when mixed with mCherry (Figure 13A). The mean fluorescence
of sfGFP individually was 168008 ± 2503.4 compared to the mean fluorescence of
sfGFP mixed with mCherry at 172678.2 ± 6826.9. Similarly, at mCherry
excitation and emission, similar results were observed and there was no
significant difference in fluorescence intensity (Figure 1B). The mean
fluorescence of mCherry individually was 4524.8 ± 5.1 compared to the mean
fluorescence of mCherry mixed with sfGFP at 4693.6 ± 162.5.

In conclusion, mixing of sfGFP and mCherry did not significantly alter the
fluorescence intensity of each individual fluorescent protein. It can be
concluded that despite the mixing of fluorescent proteins, when measuring
fluorescence intensity, it can be confidently assumed that the results for
each fluorescent protein is accurate.

For more detail see -
<https://2019.igem.org/Team:Newcastle/Results/bronzecharacterisation>

  

## Team Aalto-Helsinki 2019 characterization: Effect of antibiotic
concentration on fluorescence intensity

#### Aim

The aim was to characterize the RFP coding device (BBa_J04450) by measuring
the effect of a range of different chloramphenicol concentrations on
fluorescence intensity in _Escherichia coli_ DH5a.

#### Method

For the experiment, a DH5a colony harboring the BBa_J04450 was used to
inoculate 20 ml of LB medium supplemented with 25 ug/ml chloramphenicol. The
culture was grown for 24 h at +37 C with 250 rpm shaking. The experiment was
carried out on a 96-well plate with 90 ul of LB+cam and 90 ul of cell culture
in each well. The following chloramphenicol concentrations were tested: 12,5
mM; 17,5 mM; 25 mM; 32,5 mM; 62,5 mM; 87,5 mM. Concentrations represent final
concentrations in each well after addition of cell culture.  
  

Measurement was carried out with Synergy H1 microplate reader (Biotek).
Excitation read was set to 556 nm and emission to 586 nm. OD600 was measured
from each well. Optics were read from the bottom of each well. Temperature was
set at 37 C (+ preheat) and the plate was under continuous shaking with 230
rpm. Fluorescence and absorbance readings were measured every 10 min over 8 h.
Mean fluorescence and OD600 values were calculated from three replicates at
each time point. Fluorescence values were normalized with OD600 values so that
differences in cell growth would be considered. As a reference for this
method, we used a protocol kindly provided by Christopher Jonkergouw from
Aalto University. The protocol can be found in our WIKI page under Protocols
(18. Fluorescence expression in Microtitre).  
  

#### Results

OD600 values for each tested antibiotic concentration are presented in Figure
1. Absolute fluorescence values are presented in Figure 2, and normalized
fluorescence values in Figure 3.

  

[![](/wiki/images/thumb/0/0e/T--Aalto-Helsinki--J04450_chart1.png/530px-T--
Aalto-Helsinki--J04450_chart1.png)](/File:T--Aalto-Helsinki--
J04450_chart1.png)

[](/File:T--Aalto-Helsinki--J04450_chart1.png "Enlarge")

Figure 1. OD600 values of DH5a harboring BBa_J04450 grown with chloramphenicol
concentrations ranging from 12,5 mM to 87,5 mM.

  
Initial OD600 levels were between 0.1-1. Cell density of cultures supplemented
with lowest antibiotic concentration (12,5) increased from 1.0 to 1.3 during 8
h incubation. Cell density of cultures with higher antibiotic concentrations
increased slightly less, up to 1.1 approximately.

  

[![](/wiki/images/thumb/b/b3/T--Aalto-Helsinki--J04450_chart2.png/530px-T--
Aalto-Helsinki--J04450_chart2.png)](/File:T--Aalto-Helsinki--
J04450_chart2.png)

[](/File:T--Aalto-Helsinki--J04450_chart2.png "Enlarge")

Figure 2. Absolute fluorescence values of DH5a harboring BBa_J04450 measured
at 556-586 nm. Cells were grown with chloramphenicol concentrations ranging
from 12,5 mM to 87,5 mM.

Fluorescence values were highest in cell cultures supplemented with lowest
antibiotic concentrations (12,5 mM), following a similar pattern to cell
density. Second highest fluorescence values were observed with cells
supplemented with 25 mM, followed by 62,5 mM concentrations.

  

[![](/wiki/images/thumb/d/d1/T--Aalto-Helsinki--J04450_chart3.png/530px-T--
Aalto-Helsinki--J04450_chart3.png)](/File:T--Aalto-Helsinki--
J04450_chart3.png)

[](/File:T--Aalto-Helsinki--J04450_chart3.png "Enlarge")

Figure 3. Normalized fluorescence values of DH5a harboring BBa_J04450 measured
at 556-586 nm. Cells were grown with chloramphenicol concentrations ranging
from 12,5 mM to 87,5 mM.

  
Normalized fluorescence values increased in all chloramphenicol concentrations
we tested. Fastest increase was observed with the lowest antibiotic
concentration. After 8 hours of incubation, highest antibiotic concentration
showed the highest fluorescence readings. This is probably because cell
density in 87,5 mM cultures increased after 6,5 h incubation.

#### In conclusion

It was shown that different chloramphenicol concentrations have an effect on
fluorescence intensity. Lower chloramphenicol concentration (12,5 mM) which is
typically applied could maintain RFP expression in DH5 for 8 h. It was also
shown that cells were able to grow and express RFP with chloramphenicol
concentration of 87,5, which is 3,5 times higher than the recommended
concentration (25 mM). Cells supplemented with 17,5 and 32,5 mM concentrations
had lowest expression of RFP of the range of concentrations studied here.

## **Team Botchan_Lab_tokyo 2019 characterization**

## **description**

We transformed the plasmid which RFP is inserted in psB1C3 into DH5α, and
tested radiation tolerance. The following graph shows the result. About
protocol, please refer to 3.2 RFP in the protocol of our that's page.

## **Experimental Design**

Gamma-irradiation and cell survival assay

We transformed DH5α with the plasmid which RFP is inserted in psB1C3 and
tested radiation tolerance.

・At first the cells are cultured in liquid medium and the medium is replaced
with Butterfield‘s Phosphate Buffer (BPB). This is because when the gamma ray
is irradiated, the complex culture medium might affect the cell and it might
disturb the results. So it‘s necessary to replace the complex and unknown
factor before gamma-irradiation.

・After replacement with BPB, irradiate a certain amount of gamma rays (0, 10,
50, 100 Gy*) and spread the solution on the plate. Incubate them at 30 ℃.

・Finally, count the number of colonies that appeared a few days later (48－72
hours) and calculate the average Colony Forming Unit (CFU/ mL, equal to the
viable cell concentration). The average CFU / mL of an irradiated sample (N)
was divided by the average CFU / mL of a non-irradiated sample (N0) to produce
the survival rate (N0/N).

  * Gy = J/kg, the SI unit of energy absorbed from ionizing radiation.

## **Result**

survival rate of _E. coli_ (have plasmid RFP(BBa_J04450)inserted)

[![T--Botchan Lab Tokyo-- R1.png](/wiki/images/thumb/7/78/T--Botchan_Lab_Tokyo
--_R1.png/600px-T--Botchan_Lab_Tokyo--_R1.png)](/File:T--Botchan_Lab_Tokyo--
_R1.png)

[](/File:T--Botchan_Lab_Tokyo--_R1.png "Enlarge")

Figure1. The viability of E. coli DH5α against gamma irradiation

_E. coli_ showed a significant decrease in viability between 50 Gy and 100 Gy
versus 10 Gy. From this result, we concluded that _E. coli_ is highly
sensitive to gamma irradiation when it is irradiated with 50 and more doses of
Gy (As for 0 Gy, an irradiated sample was as same as a non-irradiated group,
so the survival rate was calculated as always 1).

## **conclusion**

This graph shows that the number of cells is significantly reduced with
respect to the intensity of radiation. That is, it was shown that E. coli
(DH5α) having a plasmid inserted with RFP has no radiation resistance.

[[edit](http://parts.igem.org/partsdb/part_info.cgi?part_name=BBa_J04450)]

Categories

//cds/reporter/chromoprotein  
//classic/reporter/pret  
//function/reporter/color  
//function/reporter/pigment

Parameters

emission| RFP| excitation| | tag| None

