---
layout: post
title:  "BBa_K914003"
author: Regulatory
categories: [collection 1 - 2021,promoter] 
image: assets/images/6.jpg
---


  * [main page](http://parts.igem.org/Part:BBa K914003)
  * [design](http://parts.igem.org/Part:BBa K914003:Design)
  * [experience](http://parts.igem.org/Part:BBa K914003:Experience)
  * [information](http://parts.igem.org/cgi/partsdb/part_info.cgi?part_name=BBa K914003)
  *   * 

[Released HQ 2013](http://parts.igem.org/Help:Part_Status_Box)

[Sample In stock](http://parts.igem.org/Help:Part_Status_Box)

Experience: Works

[33 Uses](http://parts.igem.org/partsdb/uses.cgi?part=BBa_K914003)

[1 Twin](http://parts.igem.org/partsdb/twin_info.cgi?part=BBa_K914003)

[ Get This Part](http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K914003)

![](http://parts.igem.org/images/partbypart/icon_regulatory.png)

Regulatory

# Part:BBa_K914003

Designed by: Denis Samuylov   Group: iGEM12_Paris_Bettencourt   (2012-09-20)

L-rhamnose-inducible promoter (pRha)

L-rhamnose-inducible promoter is capable of high-level recombinant protein
expression in the presence of L-rhamnose, it is also tightly regulated in the
absence of L-rhamnose by the addition of D-glucose. Read about this biobrick
part [in the context of the
project](http://2012.igem.org/Team:Paris_Bettencourt/Restriction_Enzyme).

#### Biology of pRha

L-rhamnose is taken up by the RhaT transport system, converted to L-rhamnulose
by an isomerase RhaA and then phosphorylated by a kinase RhaB. Subsequently,
the resulting rhamnulose-1-phosphate is hydrolyzed by an aldolase RhaD into
dihydroxyacetone phosphate, which is metabolized in glycolysis, and
L-lactaldehyde. The latter can be oxidized into lactate under aerobic
conditions and be reduced into L-1,2-propanediol under unaerobic conditions.

[![The E. coli rhaBRS
locus](/wiki/images/thumb/9/90/Paris_Bettencourt_2012_Prha.png/400px-
Paris_Bettencourt_2012_Prha.png)](/File:Paris_Bettencourt_2012_Prha.png)

[](/File:Paris_Bettencourt_2012_Prha.png "Enlarge")

**The _E. coli_ rhaBRS locus.** In the presence of L-rhamnose, RhaR activates
transcription of _rhaR_ and _rhaS_ , resulting in an accumulation of RhaS.
RhaS then acts as the L-rhamnose-dependent positive regulator of the _rhaB_
promoter.

The genes rhaBAD are organized in one operon which is controlled by the
rhaPBAD promoter. This promoter is regulated by two activators, RhaS and RhaR,
and the corresponding genes belong to one transcription unit which is located
in opposite direction of rhaBAD. If L-rhamnose is available, RhaR binds to the
rhaPRS promoter and activates the production of RhaR and RhaS. RhaS together
with L-rhamnose in turn binds to the rhaPBAD and the rhaPT promoter and
activates the transcription of the structural genes. However, for the
application of the rhamnose expression system it is not necessary to express
the regulatory proteins in larger quantities, because the amounts expressed
from the chromosome are sufficient to activate transcription even on multi-
copy plasmids. Therefore, only the rhaPBAD promoter has to be cloned upstream
of the gene that is to be expressed. Full induction of rhaBAD transcription
also requires binding of the CRP-cAMP complex, which is a key regulator of
catabolite repression.

The pRha sequence was containing an EcoRI restriction site, so we had to
disrupt it in order to use pRha as a biobrick. In order to decide which base
pair to modify, we used the [UCSC Microbial Genome
Browser](http://microbes.ucsc.edu). We compared the pRha sequence in E.coli
and similar species, and identified that the at the position is sometimes
replaced by a in some species, so we decided to replace it in a same way. We
ordered a gBlock with the pRha sequence having the mutation, and this is the
sequence we used and submitted.

## Characterization of pRha

#### Experimental setup

In order to characterize this promoter, we made a construct with a medium RBS
([B0032](http://parts.igem.org/Part:BBa_B0032)) and an RFP cloned downstream
of the pRha, on the pSB3C5 plasmid. We induced the expression of RFP by adding
L-Rhamnose. As a negative control, we used cells without the inducer, as well
as cells repressed with Glucose.

#### Results

First, by simple observation under a fluorescence viewer, we have seen that
the addition of 1% L-Rhamnose leads to a significant expression of RFP after
10hours. The negative controls, where no Rhamnose was added, or when the
promoter was repressed by Glucose, did not show any visible fluorescence. Both
photos are taken after we centrifuged a culture of NEB Turbo strain with
transformed plasmid. For the fluorescent result, the same tubes were
photographed under excitation light (540nm), through an emission filter
(590nm).

[![](/wiki/images/thumb/6/60/450px-
Paris_Bettencourt_2012_RG_pRha_photo_1.jpg/250px-450px-
Paris_Bettencourt_2012_RG_pRha_photo_1.jpg)](/File:450px-
Paris_Bettencourt_2012_RG_pRha_photo_1.jpg)

[](/File:450px-Paris_Bettencourt_2012_RG_pRha_photo_1.jpg "Enlarge")

_E.Coli_ **The left tube:** cells were grown without L-rhamnose. **The right
tube:** cells are grown in the present of 1% of L-rhamnose.

|

[![](/wiki/images/1/1c/250px-
Paris_Bettencourt_2012_RG_pRha_photo_2.jpg)](/File:250px-
Paris_Bettencourt_2012_RG_pRha_photo_2.jpg)

[](/File:250px-Paris_Bettencourt_2012_RG_pRha_photo_2.jpg "Enlarge")

_E.Coli_ **The right tube** which was induced by L-rhamnose expresses RFP,
while **the left tube** where we didn't add it, has no visible expression.  
  
---|---  
  
We quantified this result in a plate reader.

[![](/wiki/images/4/46/Paris_Bettencourt_2012_RG_pRha_graph_4.jpg)](/File:Paris_Bettencourt_2012_RG_pRha_graph_4.jpg)

[](/File:Paris_Bettencourt_2012_RG_pRha_graph_4.jpg "Enlarge")

Quantification of the fluorescence after 10h of growth  
  
---  
  
Next, we characterized the pRha promoter using a plate reader. We used
different concentrations of L-Rhamnose (0.05%, 0.1%, 0.2%, 0.5% and 1%) and
observed the resulting fluorescence over time. As negative controls, we used
the non-induced cells, as well as cells repressed by 1% Glucose.

[![](/wiki/images/a/a7/Paris_Bettencourt_2012_RG_pRha_graph_3.jpg)](/File:Paris_Bettencourt_2012_RG_pRha_graph_3.jpg)

[](/File:Paris_Bettencourt_2012_RG_pRha_graph_3.jpg "Enlarge")

Characterized the pRha promoter using a plate reader

As we can see from the graph above, pRha promoter works as expected and it
could be well tuned by concentration of L-rhamnose.

  

  
The UChicago 2015 iGEM team also used this part in their project and did some
characterization of the promoter, using Western blot for their gene of
interest, providing an improvement on the characterization accomplished by the
2012 Paris-Bettencourt team which used a plate-reader to measure fluorescence.
In contrast to the low copy plasmid used by the Paris Bettencourt team, they
characterized the promoter when expressed in a high copy plasmid. Accordingly,
they discovered that despite following a similar protocol (10 hours of
induction), significantly less rhamnose was required for the induction of the
promoter in the high copy number plasmid.

[![](/wiki/images/thumb/9/9e/UChicago2015Image002.png/530px-
UChicago2015Image002.png)](/File:UChicago2015Image002.png)

[](/File:UChicago2015Image002.png "Enlarge")

Induction for the characterization curve was accomplished in M9 minimal media
with supplements (0.4%glycerol, 0.1% casamino acids).

The UChicago team also verified that the pRha promoter is quite tight and has
a relatively gradual response curve, characteristics that makes the promoter
especially well-suited for assaying a wide range of concentrations of a
protein of interest.

The densitometry results from Western blots indicate the tightness of pRha. At
0% L-rhamnose, there is significantly less KaiA expressed compared to KaiB and
KaiC on a separate constitutively expressed promoter. This supports our
ability to tune KaiA concentrations to optimize oscillations.

[![](/wiki/images/thumb/9/90/Image003.jpg/530px-
Image003.jpg)](/File:Image003.jpg)

[](/File:Image003.jpg "Enlarge")

Fitted exponential curve plotted in Prism to %Rhamnose vs. ng of KaiA gradient
curve.

  
The gradient curve developed portrays the ability to fine tune the
concentrations of KaiA. The data in this graph was corrected using ng from
protein standards (below)

[![](/wiki/images/b/b2/Image004.jpg)](/File:Image004.jpg)

[](/File:Image004.jpg "Enlarge")

Fitted standard curve of purified Kai A proteins to determine ng of protein
from intensity readings in Image J.

Sequence and Features

  

Assembly Compatibility:

  * 10

COMPATIBLE WITH RFC[10]

  * 12

COMPATIBLE WITH RFC[12]

  * 21

COMPATIBLE WITH RFC[21]

  * 23

COMPATIBLE WITH RFC[23]

  * 25

COMPATIBLE WITH RFC[25]

  * 1000

COMPATIBLE WITH RFC[1000]

  

  

## Functional Parameters: Austin_UTexas

BBa_K914003 parameters

### Burden Imposed by this Part:

![](http://parts.igem.org/wiki/images/f/fa/T--Austin_Utexas--
no_burden_icon.png)

**Burden Value: 2.0 ± 7.0%**

Burden is the percent reduction in the growth rate of _E. coli_ cells
transformed with a plasmid containing this BioBrick (± values are 95%
confidence limits). This BioBrick did not exhibit a burden that was
significantly greater than zero (i.e., it appears to have little to no impact
on growth). Therefore, users can depend on this part to remain stable for many
bacterial cell divisions and in large culture volumes. Refer to any one of the
[BBa_K3174002](http://parts.igem.org/Part:BBa_K3174002) \-
[BBa_K3174007](http://parts.igem.org/Part:BBa_K3174007) pages for more
information on the methods, an explanation of the sources of burden, and other
conclusions from a large-scale measurement project conducted by the [2019
Austin_UTexas team](http://2019.igem.org/Team:Austin_UTexas).

This functional parameter was added by the [2020 Austin_UTexas
team.](http://2020.igem.org/Team:Austin_UTexas/Contribution)

## Team Aix-Marseille 2020 : Bibliographic research to ameliorate the _P
rhaBAD_ biobrick efficiency

#### RhaS

When using this promoter as a biosensor, it is used with only one gene
encoding a fluorescent protein ( _rfp_ , _gfp_ ).This is because the
Paris_Bettencourt 2012 team thought that it was sufficient to have a good
expression by rhamnose. But we learned that if we add in the plasmid a gene
that encodes constitutively for RhaS, we could have a better fluorescence.1 We
could have decoupled the RhaS protein to the RhaR protein, thus suppressing
the activator dependency.1 More, it allows us to import this biobrick in other
bacteria than _E.coli_ which don’t have _rhaS_ gene in their chromosome. This
modification has already been tested in cyanobacteria which is a clade within
it’s difficult to find a good inducible promoter system, which is no toxic,
with a linear expression.2 A study shows that _Synechocystis sp_ , a
cyanobacteria, has efficiently produced protein without either an _E.coli_
sigma factor or a protein homologous to RhaS.2

#### RhaB/RhaT

However, the efficiency of the _P rhaBAD_ system can be increased yet.

We know for exemple that the rhamnose is metabolized by _E.coli_ , so the
quantity in the cell decreases with time which involves that fluorescent
signal being transient in function of time.

So for the chassis, we can use a bacteria where the chromosomic _rhaB_ is
inactivated by introduction of a shift mutation in _rhaB_.3 Thus, L-rhamnose
isn’t metabolized anymore.3 This modification allows to have a signal that
isn’t transient anymore for the protein production but neither dose
dependant.3 The _rhaB_ mutation alone allows us to have a response « all or
nothing » independent of catabolism.3

Another point to ameliorate is the false dose-dependant effect of the _P
rhaBAD_ biobrick.3 In fact, some studies show that effect is due to the
consumption of L-rhamnose by metabolism and not due to the regulation of
production yield.3

Thus, to increase the accuracy of fluorescence measures, we can mutate _rhaB_
and also suppress the active transport of rhamnose done by RhaT by suppressing
its gene.3 Thus the signal will be only dependent on the passive diffusion of
rhamnose and so dependent on the media concentration of rhamnose.3 These two
mutations allow us to have a real accurate dose-dependant effect in fonction
of the concentration of rhamnose in media and independent of the catabolism.3

These modifications have the advantage to not decrease the growth of the
bacteria.3 But they allow overall to know exactly which concentration of
inducer is optimal to produce a chosen protein efficiently.3

The double mutation described here allows also to have a better yield of
secretory protein production which can have a saturation at the level of their
biogenesis pathway.3

#### New inducer : L-mannose

This promoter can be induced by an analogue of L-rhamnose, the L-mannose,
which has a close effect but which isn’t metabolised.1 The article cited
earlier tells us that the L-mannose could be a perfect analogue.1 Our system
described previously is sometimes even more sensitive for it than for
L-rhamnose.1 It also has the same linear response as with the L-rhamnose.1 So
we have a response that continues in time and is more powerful.1 That surely
because L-mannose isn’t well metabolized by _E.coli_ and because it is
recognized by RhaS.1 We also have an unimodal induction like with L-rhamnose
which is useful to produce the same concentration of protein in different
cells, except for low inducer concentration where the expression is bimodal.1
This bimodality at low concentration is also encountered with the Plac/IPTG
inducible system which is hugely used, so it isn’t really a problem.1 It can
be resolved by the use of another analogue, the L-lyxose, less efficient than
the L-mannose but unimodal at low concentration.1 More, these two analogues
are easily found in stores.1 The problem can however be the price of
L-mannose, relatively expensive.1

  1. Kelly et al, 2016 [Synthetic Chemical Inducers and Genetic Decoupling Enable Orthogonal Control of the rhaBAD Promoter](https://pubs.acs.org/doi/10.1021/acssynbio.6b00030)
  2. Kelly et al, 2018 [A Rhamnose-Inducible System for Precise and Temporal Control of Gene Expression in Cyanobacteria](https://pubs.acs.org/doi/10.1021/acssynbio.7b00435)
  3. Hjelm, 2017 [Tailoring Escherichia coli for the L‑Rhamnose PBAD Promoter-Based Production of Membrane and Secretory Proteins](https://pubmed.ncbi.nlm.nih.gov/28226208/)

[[edit](http://parts.igem.org/partsdb/part_info.cgi?part_name=BBa_K914003)]

Categories

//promoter

Parameters

None|

