---
layout: post
title:  "BBa_K581006"
author: RNA
categories: [collection 1 - 2021,chassis, prokaryote, ecoli, transcriptional] 
image: assets/images/6.jpg
---


  * [main page](http://parts.igem.org/Part:BBa K581006)
  * [design](http://parts.igem.org/Part:BBa K581006:Design)
  * [experience](http://parts.igem.org/Part:BBa K581006:Experience)
  * [information](http://parts.igem.org/cgi/partsdb/part_info.cgi?part_name=BBa K581006)
  *   * 

[Released HQ 2013](http://parts.igem.org/Help:Part_Status_Box)

[Sample In stock](http://parts.igem.org/Help:Part_Status_Box)

Experience: Works

Not Used

[ Get This Part](http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K581006)

![](http://parts.igem.org/images/partbypart/icon_rna.png)

RNA

# Part:BBa_K581006

Designed by: Qin Xiao   Group: iGEM11_Peking_Signaling   (2011-09-26)

Pc+ptsG(wt)-GFP (ptsG(wt) 5' UTR fused with gfp)

  

## Background

PtsG is a glucose permease which is subordinate to phosphotransferase system
and serves as a transporter. Here,we studied this mRNA perform the conjugate
part of the small RNA regulator sgrS(wt)[1]. ptsG mRNA is regulated by SgrS by
short, imperfect base-pairing interactions, and its expression is thus
repressed(See Fig.1).

[![123.png](/wiki/images/thumb/8/81/123.png/600px-123.png)](/File:123.png)

_Figure 1: Sequence alignment of wildtype ptsG/SgrS pair and its mutant
complementary pairs._

Teppei Morita et.al’ s work suggests that two mutations (C85G and C87G) in
ptsG mRNA could completely impair the ability of SgrS to downregulate its
expression, while compensatory mutations of SgrS (G178C and G176C) restore the
gene silencing ability. These results indicate that it is the base pairing of
the two RNAs rather than particular nucleotides that is important for SgrS
action. They have also illustrated that sequence outside this region, even
though complementary, is rather dispensable for the efficient silencing
(Kawamoto et al., 2006). This makes mutant ptsG/SgrS pairs orthogonal to
genetic context of the host cell. Therefore we choose this couple of conjugate
mRNA/sRNA as the foundation of our comparator device design.

By employing two sets of mutant ptsG mRNA as well as its complementary SgrS in
the design shown in Fig 1, we set to biologically implement the comparator. In
detail, ptsG1 refers to a C85G mutant of ptsG (wt) while ptsG2 is a C87G
mutant. SgrS1 (G178C) and SgrS2 (G176C) are the corresponding revertants which
could help restore their complementarity. And as a proof-of-concept
experiment, we constructed synthetic gene circuits, in which the 5’
untranslated region of ptsG mRNA was translationally fused to the coding
sequence of the reporter gfp (Levine et al., 2007), as shown in Fig 2.

[![Q Induce ptsG.png](/wiki/images/thumb/2/29/Q_Induce_ptsG.png/400px-
Q_Induce_ptsG.png)](/File:Q_Induce_ptsG.png)

[![Q Induce SgrS.png](/wiki/images/thumb/a/a2/Q_Induce_SgrS.png/400px-
Q_Induce_SgrS.png)](/File:Q_Induce_SgrS.png)

_Fig. 2 The modular components of the comparator._ _(A) Salicylate leads to
the transcription of ptsG-gfp mRNA, which is the target of constitutively
expressed SgrS. This is how we implemented both reporting and repressing
outputs as a result of the activation of Psal. When there is more salicylate
in the media, the GFP fluorescence intensity is expected to be stronger. (B)
Salicylate leads to the transcription of SgrS, while the ptsG-gfp mRNA is
downstream a constitutive promoter. In this scenario, as the concentration of
salicylate increases, the repression effect SgrS exerts on ptsG would in turn
be stronger, so the GFP fluorescence intensity is supposed to be weaker._

Furthermore, as a proof-of-concept experiment, we constructed synthetic gene
circuits, in which the 5’ untranslated region of ptsG mRNA was translationally
fused to the coding sequence of the reporter gfp. The fluorescence intensity
of GFP could reflect the repression effect that SgrS exerts on ptsG.

## Experimental Data

To qualitatively and quantitatively characterize the performance of our
competitor, we conducted the following experiments.

**Part I. The Orthogonal Silencing Matrix**

The repression capacity of each ptsG/SgrS pair was indicated by the ratio of
the average fluorescence intensity before to after the trigger of SgrS. What
we expected was a significant repression within the cognate pairs
(ptsG1/SgrS1, ptsG2/SgrS2, and ptsG (wt)/SgrS (wt)), and a minor repression
folds among different pairs.

As Figure 3 shows, the highest ratio lie at the diagonal from the upper left
to the lower right as expected, which is 5 to 6 folds. As for the ptsG
(wt)/SgrS1&2, ptsG1/SgrS (wt), and ptsG2/SgrS (wt), given that these crosses
differ at only one base pair, the repression efficacy is around 3 folds. By
contrast, the inhibiting effect of on ptsG2 and SgrS2 on ptsG1 is rather
unapparent, which can be seen as an appropriate characteristic fitting our
competitor requirements.
[![Matrix.jpg](/wiki/images/7/7a/Matrix.jpg)](/File:Matrix.jpg)

**Part II. The Response Curve**

Levine et. al’s work has revealed that in the idealized scenario when binding
between sRNA and mRNA occurs extremely rapidly, gene expression is completely
silenced if the target transcription rate is below a threshold. Above this
threshold, gene expression will increase linearly. Such threshold-linear model
is based on the difference of transcription rates between sRNA and mRNA(Levine
et al., 2007).

Our data fitted with the predicted scenario quite readily, just as Fig 4
shows.

  

[![Def.png](/wiki/images/thumb/5/53/Def.png/600px-Def.png)](/File:Def.png)

_Fig. 4 The dose-response curve of SgrS/ptsG-gfp interaction. (A) Salicylate-
induced SgrS repressing the expression of ptsG-GFP. The promoter activity is
defined as the GFP expression of the Psal+gfp strain grown in identical media.
Different promoter activities were obtained by varying salicylate
concentration in the media. The conjugate pairs fit into dose-response curve
with variable Hill slope given as a parameter, and the R^2 is 0.9974, 0.9879,
and 0.9607 corresponding to ptsG(wt)/SgrS(wt), ptsG1/SgrS1 and ptsG2/SgrS2,
respectively. (B) When sgrS (wt) was constitutively expressed and ptsG (wt)
salicylate-induced, a dose-response curve is also fitted with an R^2 of
0.9301._

Such results are in accordance with Levine et. al’s conclusion, i.e., the
binding rates between mRNA and sRNA in effect are inherently limited, so the
threshold-linear model couldn’t be strictly fitted (Levine et al., 2007). But
the performance of SgrS/ptsG pairs is very close to the idealized threshold-
linear mode of regulation.

## Reference

[1] Geissmann, T.A., and Touati, D. (2004). Hfq, a new chaperoning role:
binding to messenger RNA determines access for small RNA regulator. _The EMBO
journal_ **23:** 396-405

[2] Kawamoto, H., Koide, Y., Morita, T., and Aiba, H. (2006). Base-pairing
requirement for RNA silencing by a bacterial small RNA and acceleration of
duplex formation by Hfq. _Molecular microbiology_ **61:** 1013-1022

[3] Levine, E., Zhang, Z., Kuhlman, T., and Hwa, T. (2007). Quantitative
characteristics of gene regulation by small RNA. _PLoS biology_ **5:** e229
Sequence and Features

Assembly Compatibility:

  * 10

COMPATIBLE WITH RFC[10]

  * 12

INCOMPATIBLE WITH RFC[12]

Illegal NheI site found at 7  
Illegal NheI site found at 30  

  * 21

COMPATIBLE WITH RFC[21]

  * 23

COMPATIBLE WITH RFC[23]

  * 25

COMPATIBLE WITH RFC[25]

  * 1000

INCOMPATIBLE WITH RFC[1000]

Illegal BsaI.rc site found at 790  

  

[[edit](http://parts.igem.org/partsdb/part_info.cgi?part_name=BBa_K581006)]

Categories

//chassis/prokaryote/ecoli  
transcriptional

Parameters

biology| 5' regulatory region of ptsG(wt) is the target of sRNA SgrS(wt)|
chassis| E. coli DH5α| n/a| Pc+ptsG(wt)-GFP (ptsG(wt) 5' UTR fused with gfp)

