---
layout: post
title:  "BBa_K731722"
author: Terminator
categories: [collection 1 - 2021,rnap, bacteriophage, t7, terminator, single] 
image: assets/images/6.jpg
---


  * [main page](http://parts.igem.org/Part:BBa K731722)
  * [design](http://parts.igem.org/Part:BBa K731722:Design)
  * [experience](http://parts.igem.org/Part:BBa K731722:Experience)
  * [information](http://parts.igem.org/cgi/partsdb/part_info.cgi?part_name=BBa K731722)
  *   * 

[Released HQ 2013](http://parts.igem.org/Help:Part_Status_Box)

[Sample In stock](http://parts.igem.org/Help:Part_Status_Box)

Experience: Works

[3 Uses](http://parts.igem.org/partsdb/uses.cgi?part=BBa_K731722)

[1 Twin](http://parts.igem.org/partsdb/twin_info.cgi?part=BBa_K731722)

[ Get This Part](http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K731722)

![](http://parts.igem.org/images/partbypart/icon_terminator.png)

Terminator

# Part:BBa_K731722

Designed by: Anna Depetris, Giacomo Giacomelli   Group: iGEM12_TRENTO
(2012-08-23)

[![Terminator.png](/wiki/images/8/81/Terminator.png)](/File:Terminator.png)

| biology| rrnBT1| direction| Forward| forward_efficiency| 0.976 +/- 0.005
with T7 RNAP (T7 promoter) 0.985 +/- 0.001 with E. coli RNAP (Ptac)

|

T1 terminator from E. coli rrnB

  * This is the [BBa_B0010](/Part:BBa_B0010 "Part:BBa B0010") part extracted by PCR from the double terminator [BBa_B0015](/Part:BBa_B0015 "Part:BBa B0015") contained in [BBa_E0840](/Part:BBa_E0840 "Part:BBa E0840").

Since many different teams had problems using [BBa_B0010](/Part:BBa_B0010
"Part:BBa B0010"), this is the same part newly submitted, confirmed by
sequencing and by experience...ready to use!

The characterization of this part was done by the Trento iGEM team 2012 using
the new platforms for terminator characterization that they have built and
submitted to the Registry as [BBa_K731700](/Part:BBa_K731700 "Part:BBa
K731700") and [BBa_K731710](/Part:BBa_K731710 "Part:BBa K731710").  
  
---|---|---  
  
**Examples of Secondary Structures**

[![B0010 1.png](/wiki/images/d/d1/B0010_1.png)](/File:B0010_1.png)[![B0010
2.png](/wiki/images/6/60/B0010_2.png)](/File:B0010_2.png)

  

Sequence and Features

  

Assembly Compatibility:

  * 10

COMPATIBLE WITH RFC[10]

  * 12

COMPATIBLE WITH RFC[12]

  * 21

COMPATIBLE WITH RFC[21]

  * 23

COMPATIBLE WITH RFC[23]

  * 25

COMPATIBLE WITH RFC[25]

  * 1000

COMPATIBLE WITH RFC[1000]

  

* * *

**Usage and Biology**

  
We made some preliminary analyses to define the best procedure to characterize
this terminator using [BBa_K731700](/Part:BBa_K731700 "Part:BBa K731700") and
[BBa_K731710](/Part:BBa_K731710 "Part:BBa K731710"). We found that the
emission from A206K Venus (i.e. mVenus) was much stronger than the mCherry
emission, resulting in some spectral overlap. To improve the quality of the
data, off-peak excitation and emission wavelengths were identified that
minimized the effect. Therefore, mVenus was excitated at 485 nm (excitation
peak is at 515 nm), and the emission of mVenus was collected at its maximum
position, i.e. 528 nm. The maximum mCherry excitation wavelength was used (587
nm), but the emission of mCherry was read at 615 nm, as opposed to the maximum
at 609 nm. Here is a summary of these wavelengths:

| Standard Excitation (nm)  | Standard Emission (nm)  | Modified Excitation
(nm)  | Modified Emission (nm)  
---|---|---|---|---  
mCherry  | 587  | 609  | 587  | 615  
mVenus  | 515  | 528  | 485  | 528  
  
_**TABLE 1.** **Standard and modified excitation and emission wavelengths**  
_

  
The activity of this part was analyzed both with T7 and _E. coli_ RNA
polymerases _in vivo_ and with T7 RNA polymerase _in vitro_. The activity of
this terminator was measured as the ratio between the two proteins' levels
using the equation found in literature for termination efficiency. Doing this
measurements, however, we realized that terminators can have an effect also on
mCherry expression, enhancing it. This effect was previously described Abe. As
a consequence of this unexpected outcome, we defined other two parameters that
are here summarized in addition to the literature definition of termination
efficiency.

The parameters used to analyze the data are:

apparent termination efficiency, calculated with the equation found in
literature Nojima,
[![FormulaEa.jpg](/wiki/images/2/29/FormulaEa.jpg)](/File:FormulaEa.jpg)

raw termination efficiency, that does not consider the mCherry contribution
[![FormulaEr.jpg](/wiki/images/9/9a/FormulaEr.jpg)](/File:FormulaEr.jpg)

relative increase in the upstream gene expression,
[![RIequation.png](/wiki/images/7/73/RIequation.png)](/File:RIequation.png)

  
where

- _Vs_ is the A206K Venus peak’s intensity of the construct with the terminator of interest inserted in the prefix-suffix linker 

- _Vc_ is the A206K Venus peak’s intensity of the control construct without intervening terminator 

- _Cs_ is the mCherry peak’s intensity of the construct with the terminator inserted 

- _Cc_ is the mCherry peak’s intensity of the control construct 

  
**_In vivo_ measurements:**

Our experiments exploited an _E. coli_ lysogen strain carrying T7 RNA
polymerase and lacIq. Additionally, the cells, i.e. _E. coli_ BL21(DE3) pLysS,
also contained a plasmid encoding T7 lysozyme and chloramphenicol resistance.
T7 lysozyme is a natural inhibitor of T7 RNA polymerase activity, thus
reducing background expression of the target genes. The T7 RNA polymerase is
behind a lacUV5 promoter.

[![Ecoliterminatorinvivo.jpg](/wiki/images/b/bd/Ecoliterminatorinvivo.jpg)](/File:Ecoliterminatorinvivo.jpg)

  

[![Ecoliterminatortabinvivo.jpg](/wiki/images/4/41/Ecoliterminatortabinvivo.jpg)](/File:Ecoliterminatortabinvivo.jpg)

_**FIGURE 1.** **_E.coli_ terminator's effect on protein expression with two
different RNA polymerases**  
The data shown in figure 1 were acquired in two different days. For each day 4
different replicates were measured at different times. Briefly, BL21(DE3)pLysS
were grown in 10 mL of LB until an OD of 0.6 was reached and induced with 0.5
mM IPTG. After 3 hours of induction, 4 separate aliquots of 1 mL were taken
and sonicated 3 times for 10 seconds at intervals of 30 seconds. After
sonication the samples were diluted 1:2 with PBS 1X directly into a cuvette
and incubated overnight at 4°C. Fluorescence measurements were taken with a
Cary Eclipse Varian fluorimeter with a window ranging from 450 nm to 700 nm, a
slit of 5nm, a voltage of 570V for the characterization with T7 polymerase and
of 520V for the measurements with _E. coli_ RNA polymerase. _

**In vitro measurements:**

We performed the in vitro analysis only with the T7 RNA polymerase.

[![Ecoliterminatorinvitro.jpg](/wiki/images/f/f2/Ecoliterminatorinvitro.jpg)](/File:Ecoliterminatorinvitro.jpg)

[![Ecoliterminatortabinvitro.jpg](/wiki/images/2/29/Ecoliterminatortabinvitro.jpg)](/File:Ecoliterminatortabinvitro.jpg)

_**FIGURE 2.** **E.coli terminator's effect on in vitro protein synthesis with
the T7 RNA polymerases**  
Cell free measurements were done with the PurExpress kit from New England
Biolabs, using 250 ng of DNA previously purified by ethanol precipitation,
following the protocol suggested by the manufacturer. Measurements were done
with a PTI fluorimeter. _

We submitted also this Part inside both [BBa_K731700](/Part:BBa_K731700
"Part:BBa K731700") (T7 promoter backbone) and [BBa_K731710](/Part:BBa_K731710
"Part:BBa K731710") ( _tac_ promoter backbone) as, respecitively,
[BBa_K731701](/Part:BBa_K731701 "Part:BBa K731701") and
[BBa_K731702](/Part:BBa_K731702 "Part:BBa K731702"). We hope that this could
be helpful for anyone who want to verify and further delve into our results.

More information about the procedure used, the results obtained and our
considerations can be found in the [Trento iGEM 2012 wiki
page](http://2012.igem.org/Team:UNITN-Trento%7C).

## References

<biblio>

  1. Nojima pmid=16379390
  2. Abe pmid=9150882

</biblio>

  

## Functional Parameters: Austin_UTexas

BBa_K731722 parameters

### Burden Imposed by this Part:

![](http://parts.igem.org/wiki/images/f/fa/T--Austin_Utexas--
no_burden_icon.png)

**Burden Value: -2.7 ± 3.6%**

Burden is the percent reduction in the growth rate of _E. coli_ cells
transformed with a plasmid containing this BioBrick (± values are 95%
confidence limits). This BioBrick did not exhibit a burden that was
significantly greater than zero (i.e., it appears to have little to no impact
on growth). Therefore, users can depend on this part to remain stable for many
bacterial cell divisions and in large culture volumes. Refer to any one of the
[BBa_K3174002](http://parts.igem.org/Part:BBa_K3174002) \-
[BBa_K3174007](http://parts.igem.org/Part:BBa_K3174007) pages for more
information on the methods, an explanation of the sources of burden, and other
conclusions from a large-scale measurement project conducted by the [2019
Austin_UTexas team](http://2019.igem.org/Team:Austin_UTexas).

This functional parameter was added by the [2020 Austin_UTexas
team.](http://2020.igem.org/Team:Austin_UTexas/Contribution)

[[edit](http://parts.igem.org/partsdb/part_info.cgi?part_name=BBa_K731722)]

Categories

//rnap/bacteriophage/t7  
//terminator/single

Parameters

biology| rrnBT1| direction| Forward| forward_efficiency| 0.976 +/- 0.005 with
T7 RNAP (T7 promoter) 0.985 +/- 0.001 with E. coli RNAP (Ptac)

