---
layout: post
title:  "BBa_K581005"
author: RNA
categories: [collection 1 - 2021,chassis, prokaryote, ecoli, regulation, negative] 
image: assets/images/6.jpg
---


  * [main page](http://parts.igem.org/Part:BBa K581005)
  * [design](http://parts.igem.org/Part:BBa K581005:Design)
  * [experience](http://parts.igem.org/Part:BBa K581005:Experience)
  * [information](http://parts.igem.org/cgi/partsdb/part_info.cgi?part_name=BBa K581005)
  *   * 

[Released HQ 2013](http://parts.igem.org/Help:Part_Status_Box)

[Sample In stock](http://parts.igem.org/Help:Part_Status_Box)

Experience: Works

Not Used

[ Get This Part](http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K581005)

![](http://parts.igem.org/images/partbypart/icon_rna.png)

RNA

# Part:BBa_K581005

Designed by: Qin Xiao   Group: iGEM11_Peking_Signaling   (2011-09-26)

Pc+sgrS(wt)+Terminator (small RNA regulator, conjugate part of ptsG(wt))

**SgrS** (sugar transport-related sRNA) **(wt)** is a small RNA regulator that
help cells recover from glucose-phosphate stress by base pairing with
**ptsG(wt)**[[1]](http://parts.igem.org/Part:BBa_K581006) mRNA. SgrS regulates
ptsG mRNA by short, imperfect base-pairing interactions; as a result, the
expression of PtsG is repressed.

Teppei Morita et.al’ s work suggests that two mutations (C85G and C87G) in
ptsG mRNA could completely impair the ability of SgrS to downregulate its
expression, while compensatory mutations of SgrS (G178C and G176C) restore the
gene silencing ability. These results indicate that it is the base pairing of
the two RNAs rather than particular nucleotides that is important for SgrS
action. They have also illustrated that sequence outside this region, even
though complementary, is rather dispensable for the efficient silencing
(Kawamoto et al., 2006). This makes mutant ptsG/SgrS pairs orthogonal to
genetic context of the host cell. Therefore we choose this couple of conjugate
mRNA/sRNA as the foundation of our comparator device design.

**SgrS(wt)** in this part will be constitutively expressed in E.coli after
transformation processing.

    
    
                     [![Mechanism.png](/wiki/images/5/5e/Mechanism.png)](/File:Mechanism.png)
                   **Fig.1** Sequence alignment of wildtype ptsG/SgrS pair and its mutant complementary pairs. 
    

  

Sequence and Features

  

Assembly Compatibility:

  * 10

COMPATIBLE WITH RFC[10]

  * 12

INCOMPATIBLE WITH RFC[12]

Illegal NheI site found at 7  
Illegal NheI site found at 30  

  * 21

COMPATIBLE WITH RFC[21]

  * 23

COMPATIBLE WITH RFC[23]

  * 25

COMPATIBLE WITH RFC[25]

  * 1000

COMPATIBLE WITH RFC[1000]

  

  

  

  

  

  

  
  
  

_Following is a contribution to this part, by the NUS_Singapore 2019 team._

  

### Group

NUS_Singapore 2019

### Summary and Uploads

Team NUS 2019 has contributed additional characterization data for biobrick
BBa_K581005 which is supposed to repress the expression of ptsG in cell. In
2007, Wadler and Vanderpool reported SgrS’ ability to inhibit growth in _E.
coli_ , especially when glucose is the only carbon source (Wadler &
Vanderpool, 2007). This inspired team NUS Singapore 2019 to explore SgrS as a
potential candidate to fine tune the growth rate of _E. coli_ MG1655 so as to
prolong the overall viable lifespan of _E. coli_.  
  
Looking at past characterization data of this biobrick which utilized a
constitutive promoter to express SgrS, team NUS 2019 adopted tetracycline
inducible system to control the expression of SgrS instead. As the team was
interested to study the effect of varying anhydrotetracycline (aTc)
concentrations on _E. coli_ MG1655 growth and protein production, we co-
transformed RFP regulated by IPTG inducible system to validate protein
production. Concentrations of aTc ranged from 0 to 1.5mM were investigated.
The cells were grown in LB media at 37°C, washed with M9 media (0% glucose)
and finally characterized in M9 with 0.2% glucose at a starting OD600 of 0.1
at 0h. Cells were induced with respective aTc concentrations at 0h and 150uM
of IPTG at 1hr, which was then read for 20h continuously using a microplate
reader to compare growth (Figure 1) and total RFP produced (Figure 2).
Furthermore, we normalized RFP to OD600 to determine the maximum RFP
production per cell (Figure 3).  
  
![](https://2019.igem.org/wiki/images/e/e2/T--NUS_Singapore--
PartsRegistry_s1.jpeg)  
_Figure 1: Growth curve of MG1655 SgrS_A6A induced with aTc (0nm, 10nm, 100nm
and 1500nm) at 0h and 150uM of IPTG at 1h_
![](https://2019.igem.org/wiki/images/0/03/T--NUS_Singapore--
PartsRegistry_s2.jpeg)  
_Figure 2: Total RFP of MG1655 SgrS_A6A induced with aTc (0nm, 10nm, 100nm and
1500nm) at 0h and 150uM of IPTG at 1h_
![](https://2019.igem.org/wiki/images/3/3b/T--NUS_Singapore--
PartsRegistry_s3.jpeg)  
_Figure 3: RFP/OD 600 of MG1655 SgrS_A6A induced with aTc (0nm, 10nm, 100nm
and 1500nm) at 0h and 150uM of IPTG at 1h_  
  
With these results, the effects of SgrS on cells affects its growth which
directly influences the protein production; with higher concentrations of aTc
producing lesser intensity of RFP and vice versa.  
  
Reference:  
  
Wadler, C. S., & Vanderpool, C. K. (2007). A dual function for a bacterial
small RNA : SgrS performs base pairing-dependent regulation and encodes a
functional polypeptide. PNAS, 104(51), 20454–20459.

  

  

  

## Functional Parameters: Austin_UTexas

BBa_K581005 parameters

### Burden Imposed by this Part:

![](http://parts.igem.org/wiki/images/f/fa/T--Austin_Utexas--
no_burden_icon.png)

**Burden Value: 3.8 ± 3.9%**

Burden is the percent reduction in the growth rate of _E. coli_ cells
transformed with a plasmid containing this BioBrick (± values are 95%
confidence limits). This BioBrick did not exhibit a burden that was
significantly greater than zero (i.e., it appears to have little to no impact
on growth). Therefore, users can depend on this part to remain stable for many
bacterial cell divisions and in large culture volumes. Refer to any one of the
[BBa_K3174002](http://parts.igem.org/Part:BBa_K3174002) \-
[BBa_K3174007](http://parts.igem.org/Part:BBa_K3174007) pages for more
information on the methods, an explanation of the sources of burden, and other
conclusions from a large-scale measurement project conducted by the [2019
Austin_UTexas team](http://2019.igem.org/Team:Austin_UTexas).

This functional parameter was added by the [2020 Austin_UTexas
team.](http://2020.igem.org/Team:Austin_UTexas/Contribution)

[[edit](http://parts.igem.org/partsdb/part_info.cgi?part_name=BBa_K581005)]

Categories

//chassis/prokaryote/ecoli  
//regulation/negative

Parameters

device_type| post-transcriptional regulators of gene expression| family| Hfq-
binding small RNA| function| destabilzes ptsG(wt) mRNA| origin| E.coli K-12
strain| target| 5' Untranslated Regions

