---
layout: post
title:  "BBa_K608012"
author: Regulatory
categories: [collection 1 - 2021,] 
image: assets/images/6.jpg
---


  * [main page](http://parts.igem.org/Part:BBa K608012)
  * [design](http://parts.igem.org/Part:BBa K608012:Design)
  * [experience](http://parts.igem.org/Part:BBa K608012:Experience)
  * [information](http://parts.igem.org/cgi/partsdb/part_info.cgi?part_name=BBa K608012)
  *   * 

[Released HQ 2013](http://parts.igem.org/Help:Part_Status_Box)

[Sample In stock](http://parts.igem.org/Help:Part_Status_Box)

Experience: None

Not Used

[ Get This Part](http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K608012)

![](http://parts.igem.org/images/partbypart/icon_regulatory.png)

Regulatory

# Part:BBa_K608012

Designed by: Sandra Wassner   Group: iGEM11_Freiburg   (2011-09-14)

Medium promoter with weak RBS and GFP

Medium promoter from the constitutive promoter family combined with a weak RBS
(PR6) and tagged with GFP to quantify the gene expression.

The GFP fluorescence was measured with a plate reader:  

[![](/wiki/images/thumb/1/17/Freiburg2011_BSA.jpg/350px-
Freiburg2011_BSA.jpg)](/File:Freiburg2011_BSA.jpg)

[](/File:Freiburg2011_BSA.jpg "Enlarge")

BSA calibration line

The fluorescence intensity and protein concentration were measured with the
FLUOstar Omega,  
which is a multi-mode microplate reader. Samples were pipetted into the
microplate and analyzed via the plate reader. In this experiment we focused on
the protein concentration and the fluorescence intensity of RFP. We measured
the protein concentration with the bradford-assay. This is a method to
determine the total protein concentration. To analyze the protein
concentration of the samples, Coomassie Brillant Blue was pippeted to each
sample. With the binding of the dye to the proteins the color changes from
dark red to blue. The more protein in the solution the more Coomassie dye can
bind to proteins and the more the color changes into blue. The absorption of
bound Coomassie dye is 595nm. The absorbance is proportional with the amount
of bound dye. With a series of Bovine Serum Albumin (BSA) measurements the
exact protein concentration of the samples can be determined. BSA acts like a
“marker” because the concentration of BSA is known and with a linear
calibration line the exact protein concentration can be detected.

  
GFP served as a reporter of expression. We wanted to know how strong the
promoter and RBS activity is. With this reporter gene it was possible to
analyze the expression via plate reader. GFP is excited at a wavelength of
509nm and has an emission of 520nm. The plate reader illuminates the samples
with a high energy xenon flash lamp. Optical filters or monochromator create
the exact wavelength. The more GFP in the sample the higher is the GFP
fluorescence intensity. The intensity is collected with the second optical
system and is detected with a side window photomultiplier tube.

[![](/wiki/images/1/16/Freiburg2011_GFP-PR6.jpg)](/File:Freiburg2011_GFP-
PR6.jpg)

[](/File:Freiburg2011_GFP-PR6.jpg "Enlarge")

GFP fluorescence intensity dependent on the strenght of promoter and RBS

**Promoter and RBS:**  
PR1: strong Promoter (J23104) strong RBS (B0034)  
PR2: strong Promoter (J23104) medium RBS (B0032)  
PR3: strong Promoter (J23104) weak RBS (B0031)  
PR4: medium Promoter (J23110) strong RBS (B0034)  
PR5: medium Promoter (J23110) medium RBS (B0032)  
**PR6: medium Promoter (J23110) weak RBS (B0031)**  

sample  | PR2  | PR3  | PR4  | PR5  | **PR6**  
---|---|---|---|---|---  
GFP fluorescence intensity | 11378.5  | 1445.0  | 4596.2  | 41221.1  |
**26922.7**  
factor  | 7.9  | 1.0  | 3.2  | 28.5  | **18.6**  
  
  
  
  
The results of this test show that PR6 has 18.6 times higher expression of GFP
in comparison with with PR3 which has the lowest expression. The fluorescence
intensity of GFP varies, and because of lack of time we could not repeat this
experiment. We have also tested the promotor and RBS activity with RFP as a
reporter and the results deviate from this experiment. So we are looking
forward to test this system another time.

  
  

# **Tongji_China 2019 Characterization**

Our team tested the GFP fluorescence intensity and positive rate of PR3, PR5
and PR6 in E. coli TOP10 with a flow cytometry fluoresce (CytoFLEX LX, Beckman
Coulter).  
**Protocol:**  
1\. The plasmids obtained from 5L,5M,5O,plate 1, are respectively used to
transform the TOP10 strain and coated on the LB plate with chloramphenicol.
The kanamycin resistant plasmid pET-28a(+) is transformed into the TOP10
strain, which is coated on the LB plate with kanamycin as a negative control.
The strain is cultured overnight at 37°C.  
2\. Pick 3 colonies each plates and culture them in 5 mL LB medium with
chloramphenicol 12 hours.  
3\. 4800 rmp/min centrifugate 10 minutes，using 1.5 mL PBS to dilute the
sediment.  
4\. Using the negative process control sample, adjust forward-scatter and
side-scatter voltages to place the strong cell peak as close to the center of
the detector range as possible.  
5\. Instrument gating should be set to ensure that no cell events are
discarded.  
6\. 95% negative control samples’ GFP B525-FITC-A values are less than 104.
Regard those samples the GFP B525-FITC-A values more than 104 are positive.  
7\. Collect at least 5000 events per sample.  
**Summary:** The results of this test show that in E. coli TOP10, PR5 has the
highest GFP fluorescence intensity and positive rate among PR2, PR5 and PR6.

[![](/wiki/images/thumb/9/98/T--Tongji_China--
BBa_K608008%2611%2612_Characterization_Fluorescence_Intensity.png/400px-T--
Tongji_China--
BBa_K608008%2611%2612_Characterization_Fluorescence_Intensity.png)](/File:T--
Tongji_China--
BBa_K608008%2611%2612_Characterization_Fluorescence_Intensity.png)

[](/File:T--Tongji_China--
BBa_K608008%2611%2612_Characterization_Fluorescence_Intensity.png "Enlarge")

Characterization result of BBa_K608008, BBa_K608011 and BBa_K608012, on
fluorescence intensity.

[![](/wiki/images/thumb/c/c0/T--Tongji_China--
BBa_K608008%2611%2612_Characterization_Positive_Rate.png/400px-T--Tongji_China
--BBa_K608008%2611%2612_Characterization_Positive_Rate.png)](/File:T--
Tongji_China--BBa_K608008%2611%2612_Characterization_Positive_Rate.png)

[](/File:T--Tongji_China--
BBa_K608008%2611%2612_Characterization_Positive_Rate.png "Enlarge")

Characterization result of BBa_K608008, BBa_K608011 and BBa_K608012, on
postive rate.

Sample  | PR2(BBa_K608008)  | PR5(BBa_K608011)  | **PR6(BBa_K608012)**  
---|---|---|---  
GFP Fluorescence Intensity | 24541.23  | 62923.91  | **41902.30**  
Fluorescence Intensity Factor  | 1.0  | 2.6  | **1.7**  
Positive Rate | 0.7563  | 0.9190  | **0.8579**  
  
  

## **Characterization: Average Fluorescence per Cell**

[![](/wiki/images/thumb/8/8c/T--FDR-HB_Peru--
BBa_K608010_and_BBa_K608012_characterization_graph_1.png/400px-T--FDR-HB_Peru
--BBa_K608010_and_BBa_K608012_characterization_graph_1.png)](/File:T--FDR-
HB_Peru--BBa_K608010_and_BBa_K608012_characterization_graph_1.png)

[](/File:T--FDR-HB_Peru--
BBa_K608010_and_BBa_K608012_characterization_graph_1.png "Enlarge")

**Figure 1.**

[![](/wiki/images/thumb/2/21/T--FDR-HB_Peru--
BBa_K608010_and_BBa_K608012_characterization_graph_2.png/400px-T--FDR-HB_Peru
--BBa_K608010_and_BBa_K608012_characterization_graph_2.png)](/File:T--FDR-
HB_Peru--BBa_K608010_and_BBa_K608012_characterization_graph_2.png)

[](/File:T--FDR-HB_Peru--
BBa_K608010_and_BBa_K608012_characterization_graph_2.png "Enlarge")

**Figure 2.**

[![](/wiki/images/thumb/b/ba/Graph_of_%22Mean_Fluorescence_Per_Cell_of_BBa_K808010_and_BBa_K608012_over_a_period_of_225_minutes%22.png/400px-
Graph_of_%22Mean_Fluorescence_Per_Cell_of_BBa_K808010_and_BBa_K608012_over_a_period_of_225_minutes%22.png)](/File:Graph_of_%22Mean_Fluorescence_Per_Cell_of_BBa_K808010_and_BBa_K608012_over_a_period_of_225_minutes%22.png)

[](/File:Graph_of_%22Mean_Fluorescence_Per_Cell_of_BBa_K808010_and_BBa_K608012_over_a_period_of_225_minutes%22.png
"Enlarge")

**Figure 3.**

**Group:** FDR-HB_Peru  
  
**Author:** Marry Xuan  
  
**Summary:** We compared parts BBa_K608010 and BBa_K608012 to test if the
strength of the RBS correlates to the average amount of fluorescence per cell.  
  
**Method:**  

  1. Dilute 10 ul of the culture (BBa_K608010 and BBa_K608012) into 90ul of M9 + Glucose (0.2% w/v) + CAS amino acids (0.2%) medium in the plate reader wells 
  2. Use a plate reader to read the optical density at 600nm (OD). 
  3. Use that data to calculate the dilution of the culture to reach an OD reading of 0.1 and dilute with medium (M9 + Glucose + CAS). 
  4. Take 200 µl of the previous dilution and add it to a well on the plate reader (fill 3 wells for both K608010 and K608012) 
  5. Fill 3 wells with blank medium. 
  6. Put the plate in the plate reader and collect the data for 345 minutes, reads both OD and Fluorescence (We used a low gain since the fluorescence was too intense). 
  7. Once the data was collected, the OD and Fluorescence data was normalized by subtracting the blank. Then to calculate the average fluorescence per cell, we used the following equation: (FLOnorm)/(ODnorm) 

  
**Results:** We found out that after 345 minutes, the average fluorescence of
the GFP with a strong RBS (BBa_K608010) was 2.18 times greater than the
average fluorescence of the GFP with a weak RBS (BBa_K608012). This data
counters the data from the original experiment performed, where the
fluorescence intensity of BBa_K608010 was 5.86 times lower than the
fluorescence intensity of BBa_K608012.

Sequence and Features

  

Assembly Compatibility:

  * 10

COMPATIBLE WITH RFC[10]

  * 12

INCOMPATIBLE WITH RFC[12]

Illegal NheI site found at 7  
Illegal NheI site found at 30  

  * 21

COMPATIBLE WITH RFC[21]

  * 23

COMPATIBLE WITH RFC[23]

  * 25

COMPATIBLE WITH RFC[25]

  * 1000

INCOMPATIBLE WITH RFC[1000]

Illegal BsaI.rc site found at 707  

  

### Functional Parameters

  

# **Team University of Edinburgh 2019**

**Group:** University of Edinburgh OG, 2019  
**Author:** Francisco Ivan Rodriguez Jaubert and Cathal O’Reilly  
  
**Summary:** We compared the amount of GFP expressed under the medium
constitutive promoter with strong, medium and weak RBS/GFP (PR4-GFP, PR5-GFP,
PR6-GFP) in TOP10, BL21 (DE3) and DH5α cells lines. We were interested in
replicating the original characterisation as the Freiburg 2011 team mentioned
it needed to be replicated. We did so with 3 biological replicates and 3
technical replicates for each control. Additionally, we characterised these
biobricks across different common E. coli strains to investigate the
consistency across these strains.  
  
**Methods**  
Biobricks were extracted and amplified from the iGEM kit 2019 following
standard protocols. Transformation was performed following heat-shock standard
protocols. Three positive colonies were picked containing each biobrick in
each different cell line. The samples were repeated in series of three to
ensure consistency. This was carried out by adding 200 μl of cell culture to
wells of a 96-well microplate. The plate reader was set at 485nm wavelength
and 520nm emission for analysis.

  
  
**Results**  

The next graph shows the average of the three samples (repeated three times)
for each biobrick (PR4-GFP, PR5-GFP, and PR6-GFP) in different strains with
their respective GFP fluorescence intensity. The weakest RBS (PR6-GFP) in BL21
presents the highest GFP absorbance, in contrast with the Freiburg data
showing that the medium RBS (PR5-GFP) had the highest GFP absorbance.

  
[![Figure 1. GFP Flourescence Activity \(PR4-GFP, PR5-GFP, and
PR6-GFP\)](/wiki/images/thumb/4/4a/T--Edinburgh_OG--
GFP_Flourescence_Activity_CellOrder2.png/400px-T--Edinburgh_OG--
GFP_Flourescence_Activity_CellOrder2.png)](/File:T--Edinburgh_OG--
GFP_Flourescence_Activity_CellOrder2.png "Figure 1. GFP Flourescence Activity
\(PR4-GFP, PR5-GFP, and PR6-GFP\)")

Our data suggests that in BL21 the highest GFP expression was led by the
weakest RBS (PR6-GFP), followed by the strongest RBS (PR4-GFP), and finally
the medium RBS (PR5-GFP).

TOP10 shows the highest GFP expression with the strongest RBS (PR4-GFP),
followed by the medium RBS (PR5-GFP), and finally the weakest RBS (PR6-GFP).

DH5a shows the highest GFP expression with the strongest RBS (PR4-GFP),
followed by the weakest RBS (PR6-GFP), and finally the medium RBS (PR5-GFP).

TOP10 is the only cell line showing consistency in the hierarchy of the
strongest to the weakest RBS.

  

  

  

## Functional Parameters: Austin_UTexas

### Burden Imposed by this Part:

![](http://parts.igem.org/wiki/images/7/7a/T--Austin_Utexas--
high_significant_burden.png)

**Burden Value: 27.1 ± 4.7%**

Burden is the percent reduction in the growth rate of _E. coli_ cells
transformed with a plasmid containing this BioBrick (± values are 95%
confidence limits). This part exhibited a significant burden. Users should be
aware that BioBricks with a burden of >20-30% may be susceptible to mutating
to become less functional or nonfunctional as an evolutionary consequence of
this fitness cost. This risk increases as they used for more bacterial cell
divisions or in larger cultures. Users should be especially careful when
combining multiple burdensome parts, as plasmids with a total burden of >40%
are expected to mutate so quickly that they become unclonable. Refer to any
one of the [BBa_K3174002](http://parts.igem.org/Part:BBa_K3174002) \-
[BBa_K3174007](http://parts.igem.org/Part:BBa_K3174007) pages for more
information on the methods and other conclusions from a large-scale
measurement project conducted by the [2019 Austin_UTexas
team.](https://2019.igem.org/Team:Austin_UTexas)

This functional parameter was added by the [2020 Austin_UTexas
team.](https://2020.igem.org/Team:Austin_UTexas/Contribution)

[[edit](http://parts.igem.org/partsdb/part_info.cgi?part_name=BBa_K608012)]

Categories

Parameters

None|

