---
layout: post
title:  "BBa_K525710"
author: Coding
categories: [collection 1 - 2021,cds, enzyme, ligase, chassis, prokaryote, ecoli, proteindomain, affinity, proteindomain, tail, rnap, bacteriophage, t7] 
image: assets/images/6.jpg
---


  * [main page](http://parts.igem.org/Part:BBa K525710)
  * [design](http://parts.igem.org/Part:BBa K525710:Design)
  * [experience](http://parts.igem.org/Part:BBa K525710:Experience)
  * [information](http://parts.igem.org/cgi/partsdb/part_info.cgi?part_name=BBa K525710)
  *   * 

[Released HQ 2013](http://parts.igem.org/Help:Part_Status_Box)

[Sample In stock](http://parts.igem.org/Help:Part_Status_Box)

Experience: Works

Not Used

[2 Twins](http://parts.igem.org/partsdb/twin_info.cgi?part=BBa_K525710)

[ Get This Part](http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K525710)

![](http://parts.igem.org/images/partbypart/icon_coding.png)

Coding

LigA

# Part:BBa_K525710

Designed by: Anna Drong   Group: iGEM11_Bielefeld-Germany   (2011-09-12)

DNA ligase from Escherichia coli (LigA) with PT7, RBS and His-tag

### Usage and Biology

Bacterial DNA ligases are important enzymes for DNA repair and replication.
They use NAD+ as a cofactor to bind hydrolysed AMP to the active site lysine
under release of NMN. AMP is used to catalyse the formation of a
phosphodiester bond in nicked DNA by transferring AMP to the 5'-phosphate
group. This results in an attack of the nick 3'-OH on the 5'-phosphoanhydride
linkage and release of free AMP:

[![Bielefeld-Germany-2011 LigA
reactionmechanism.png](/wiki/images/thumb/b/b4/Bielefeld-
Germany-2011_LigA_reactionmechanism.png/350px-Bielefeld-
Germany-2011_LigA_reactionmechanism.png)](/File:Bielefeld-
Germany-2011_LigA_reactionmechanism.png)

During the reaction mechanism the DNA ligases functional domains undergo
conformational changes promoting intra- and intermolecular interactions. The
domains are highly conserved throughout the bacterial kingdom and categorized
in the following way:  

  * Domain 1: Ia, Ib (NTase)  

  * Domain 2: Oligomer-binding (OB) beta-barrel  

  * Domain 3: Zinc (Zn)-finger, helix-hairpin-helix motif (HhH)  

  * Domain 4: BRCT domain   

The NAD+-dependent DNA ligase from _E. coli_ (LigA) can be applied to
biochemical studies ranging from cloning procedures and enzymatic assays to
biomedical investigations. For instance, the purified enzyme can contribute to
_in vitro_ experiments dealing with antibiotic drug design because bacterial
DNA ligases are essential for fundamental life-sustaining processes and
differentiate from eukaryotic DNA ligases due to their specifity for NAD+ as a
cofactor.  

  *     * Allergen characterization of BBa_K525710: Not a potential allergen 

The Baltimore Biocrew 2017 team discovered that proteins generated through
biobrick parts can be evaluated for allergenicity. This information is
important to the people using these parts in the lab, as well as when
considering using the protein for mass production, or using in the
environment. The allergenicity test permits a comparison between the sequences
of the biobrick parts and the identified allergen proteins enlisted in a data
base.The higher the similarity between the biobricks and the proteins, the
more likely the biobrick is allergenic cross-reactive. In the full-length
alignments by FASTA, 30% or more amount of similarity signifies that the
biobrick has a Precaution Status meaning there is a potential risk with using
the part. A 50% or more amount of identity signifies that the biobrick has a
Possible Allergen Status. In the sliding window of 80 amino acid segments,
greater than 35% signifies similarity to allergens. The percentage of
similarity implies the potential of harm biobricks’ potential negative impact
to exposed populations. For more information on how to assess your own
biobrick part please see the “Allergenicity Testing Protocol” in the following
page <http://2017.igem.org/Team:Baltimore_Bio-Crew/Experiments>

For the biobrick Part:BBa_K525710, there was a 0% of identity match and 0%
similarity match to the top allergens in the allergen database. This means
that the biobrick part is not of potential allergen status. In 80 amino acid
alignments by FASTA window, no matches found that are greater than 35% for
this biobrick. This also means that there is not of potential allergen status.

### Important parameters

**Table 1: Parameters for NAD +-dependent DNA ligase from _E. coli_ (LigA).**
The parameters number of amino acids, molecular weight, theoretical pI and
extinction coefficient were calculated with the help of the [ExPASy
Translate](http://web.expasy.org/translate/) tool and the [Expasy
ProtParam](http://web.expasy.org/protparam/) tool.

Experiment  | Characteristic  | Result  
---|---|---  
Expression  
Compatibility  | _E. coli_ KRX  
Promoter  | PT7  
Inductor of expression  | L-rhamnose for induction of T7 polymerase  
Optimal temperature  | 37 °C  
Purification  
Number of amino acids  | 678  
Molecular weight  | 74.59 kDa  
Theoretical pI  | 5.59  
Extinction coefficient at 280 nm  
measured in water [M-1 cm-1]  | 37400 (assuming all pairs of Cys residues form
cystines)

36900 (assuming all Cys residues are reduced)  
  
Tag  | C-terminal 6xHis  
NAD+ detection  
Limit of detection  | 2 nM  
  
  

### Sequence and Features

Sequence and Features

  

Assembly Compatibility:

  * 10

COMPATIBLE WITH RFC[10]

  * 12

COMPATIBLE WITH RFC[12]

  * 21

INCOMPATIBLE WITH RFC[21]

Illegal BglII site found at 1449  

  * 23

COMPATIBLE WITH RFC[23]

  * 25

COMPATIBLE WITH RFC[25]

  * 1000

INCOMPATIBLE WITH RFC[1000]

Illegal SapI.rc site found at 1663  

  

  

### Purification and identification of LigA

Expression of LigA in _E. coli_ is an easy approach to produce the recombinant
protein LigA in large scale. The protein was overexpressed in _E. coli_ KRX
after induction of T7 polymerase by supplementation of 0.1 % rhamnose. Cell
harvest was performed after 4 h further growth at 37 °C. After enzymatic cell
lysis the protein could be purified with the help of Ni-NTA columns utilizing
the proteins C-terminal His-tag. As shown in figure 1 the majority was eluted
with 100 mM imidazole. The gel bands were analysed by MALDI-TOF MS and the
comparison with the Swiss-Prot database clearly identified the purified
protein as LigA (figure 2, table 2).

[![](/wiki/images/thumb/8/84/Bielefeld-Germany-2011_LigA-
purification.png/600px-Bielefeld-Germany-2011_LigA-
purification.png)](/File:Bielefeld-Germany-2011_LigA-purification.png)

[](/File:Bielefeld-Germany-2011_LigA-purification.png "Enlarge")

**Figure 1: SDS-PAGE analysis of LigA after His-tag purification.** The
cleared lysate from
[BBa_K525710](http://parts.igem.org/wiki/index.php/Part:BBa_K525710)
overexpressing _E. coli_ KRX cells was loaded on a HisTrap™ FF crude column (1
mL), the protein was eluted with imidazole and collected in 1 mL fractions.
The expected molecular weight is about 75 kDa. M: [prestained protein
marker](http://www.fermentas.de/admin/images/media/labaid_pageruler_L7P.pdf).

[![](/wiki/images/thumb/2/29/Bielefeld-Germany-2011_LigA-
purification_MALDI.png/600px-Bielefeld-Germany-2011_LigA-
purification_MALDI.png)](/File:Bielefeld-Germany-2011_LigA-
purification_MALDI.png)

[](/File:Bielefeld-Germany-2011_LigA-purification_MALDI.png "Enlarge")

**Figure 2: SDS-PAGE analysis of LigA after His-tag purification as a
preparation for MALDI-TOF MS analysis.** The cleared lysate from
[BBa_K525710](http://parts.igem.org/wiki/index.php/Part:BBa_K525710)
overexpressing _E. coli_ KRX cells was loaded on a HisTrap™ FF crude column (1
mL), the protein was eluted with 100 mM imidazole and collected in 1 mL
fractions. The framed and numbered fields indicate the gel bands which were
analysed by MALDI-TOF MS. M: [prestained protein
marker](http://www.fermentas.de/admin/images/media/labaid_pageruler_L7P.pdf).

  

**Table 2: Identification of LigA by MALDI-TOF MS.** The values correspond to
the framed and numbered gel bands in figure 2. The threshold for significance
of the Mascot Score for MS is 63 and the one for MS/MS is 26. The MS-Coverage
represents the sequence coverage of the investigated protein with the
corresponding entry in the _E. coli_ Swiss-Prot data base. The Sequence-
Coverage shows the percentage of similarity to the translated BioBrick
[BBa_K525710](http://parts.igem.org/wiki/index.php/Part:BBa_K525710)
(translation was perfomed with the [ExPASy
Translate](http://web.expasy.org/translate/) tool).

Number  | Method  | Swiss-Prot  
number  | Protein  | Protein  
Mascot  
Score  | Protein MW  | pI-Value  | MS-  
Coverage [%]  | Sequence-  
Coverage [%]  
---|---|---|---|---|---|---|---|---  
1  | MS  | B1XA82  | DNA ligase  
OS=Escherichia coli  
GN=ligA  | 96  | 73560  | 5.3  | 17  | 25.8  
2  | MS  | B1XA82  | DNA ligase  
OS=Escherichia coli  
GN=ligA  | 109  | 73560  | 5.3  | 21  | 28.7  
3  | MS  | A7ZPL2  | DNA ligase  
OS=Escherichia coli  
GN=ligA  | 41  | 73618  | 5.2  | 7  | 10.2  
4  | MS  | B1XA82  | DNA ligase  
OS=Escherichia coli  
GN=ligA  | 134  | 73560  | 5.3  | 29  | 36.6  
1  | MS/MS  | Q0TF55  | DNA ligase  
OS=Escherichia coli  
GN=ligA  | 54  | 73602  | 5.2  | 6  | /  
2  | MS/MS  | Q0TF55  | DNA ligase  
OS=Escherichia coli  
GN=ligA  | 63  | 73602  | 5.2  | 6  | /  
3  | MS/MS  | Q0TF55  | DNA ligase  
OS=Escherichia coli  
GN=ligA  | 33  | 73602  | 5.2  | 2  | /  
4  | MS/MS  | Q0TF55  | DNA ligase  
OS=Escherichia coli  
GN=ligA  | 36  | 73602  | 5.2  | 2  | /  
  
As it is reported in literature the majority of LigA usually occurs in its
adenylated form after extraction from _E. coli_. An indication for this is the
double band on the right edge of figure 2. The proteins in both gel bands were
identified as LigA from _E. coli_ suggesting that gel band 3 is the adenylated
form and gel band 4 is the deadenylated form. Depending on whether the protein
should be in its deadenylated form for further applications a treatment with
an excess of nicotinamide mononucleotide (NMN) for deadenylation is possible.

### NAD+ detection

LigA can be applied to a molecular beacon based bioassay detecting NAD+ even
in very low concentrations. For this two important requirements have to be
fulfilled: the purified LigA has to be in its deadenylated form so that DNA
ligation occurs only in presence of NAD+. Furthermore, molecular beacons have
to be designed appropriately so that they stay in their closed state under the
chosen conditions and after hybridization to a split target. Not before LigA
ligates the split target in a NAD+-dependent manner the molecular beacon is
supposed to form its open state resulting in an increase of fluorescence
intensity.  
The proof for a valid NAD+ detection system which has been used for
characterization of LigA is shown in figures 3 and 4. The molecular beacon
formed its open state after hybridization to a complementary target (not
split) so that the close proximity of the fluorophor and the quencher broke
up. After excitation with light a 45.52-fold increase in fluorescence
intensity compared to the background signal (molecular beacon alone) could be
determined. By contrast, this was not the case after using a complementary
split target hybridzied to the molecular beacon because the signal-to-
background ratio was just 3.36. This shows that the split target was not able
to induce the molecular beacons open state, but the complete target did so.
Thus, this approach is useful to measure LigA activity and therefore detect
NAD+.

  

[![](/wiki/images/thumb/b/b7/Bielefeld-Germany-2011_molecularbeacon-
target.png/650px-Bielefeld-Germany-2011_molecularbeacon-
target.png)](/File:Bielefeld-Germany-2011_molecularbeacon-target.png)

[](/File:Bielefeld-Germany-2011_molecularbeacon-target.png "Enlarge")

**Figure 3: Signal-to-background ratio (S/B) determination of 6-FAM labeled
molecular beacon in presence of a complementary target.** Calculated S/B:
45.52 (n = 3).

[![](/wiki/images/thumb/8/8e/Bielefeld-Germany-2011_molecularbeacon-
split_target.png/650px-Bielefeld-Germany-2011_molecularbeacon-
split_target.png)](/File:Bielefeld-Germany-2011_molecularbeacon-
split_target.png)

[](/File:Bielefeld-Germany-2011_molecularbeacon-split_target.png "Enlarge")

**Figure 4: Signal-to-background ratio (S/B) determination of 6-FAM labeled
molecular beacon in presence of a complementary split target.** Calculated
S/B: 3.36 (n = 3).

  
For the NAD+ bioassay the molecular beacon was preincubated with the split
target at 37 °C. After the fluorescence intensity reached equilibrium purified
and deadenylated LigA was added in a final concentration of 5 ng/µL followed
by NAD+ addition (figure 5). First of all, there was only little increase in
fluorescence intensity after LigA addition indicating that the pretreatment
with NMN for deadenylation was successful. In a separate experiment with LigA
that was not pretreated with NMN a continous increase in fluorescence
intensity was observed (data not shown). This leads to the suggestion that an
excess of NMN supports deadenylation reaction of LigA which makes the enzyme
suitable for NAD+ detection. After NAD+ addition the fluorescence intensity
increased distinctly while the fluorescence enhancement rate was dependent on
the employed NAD+ concentration. This phenomenon is illustrated in figure 6
which describes the fluorescence increase after normalization of all
fluorescence values to the first measured value for each measurement series
(NAD+ concentration) from figure 5.

  

[![](/wiki/images/thumb/5/55/Bielefeld-Germany-2011_NAD%2B_bioassay.png/650px-
Bielefeld-Germany-2011_NAD%2B_bioassay.png)](/File:Bielefeld-
Germany-2011_NAD%2B_bioassay.png)

[](/File:Bielefeld-Germany-2011_NAD%2B_bioassay.png "Enlarge")

**Figure 5: NAD + can be detected utilizing the LigA NAD+-dependence.** For
this a NAD+ bioassay was applied using 250 nM molecular beacon hybridized to a
split target. The purified LigA was pretreated with NMN for deadenylation and
added at a final concentration of 5 ng/µL. After NAD+ addition in different
concentrations (not completely listed) the target got ligated so that the
molecular beacon formed its open state resulting in an increase of
fluorescence intensity (n = 4).

[![](/wiki/images/thumb/c/cc/Bielefeld-
Germany-2011_NAD%2B_bioassay_2.png/650px-Bielefeld-
Germany-2011_NAD%2B_bioassay_2.png)](/File:Bielefeld-
Germany-2011_NAD%2B_bioassay_2.png)

[](/File:Bielefeld-Germany-2011_NAD%2B_bioassay_2.png "Enlarge")

**Figure 6: The initial fluorescence enhancement rate is dependent on the NAD
+ concentration.** For illustration the fluorescence intensity is normalized
for each measurement series to the first measured fluorescence value from
figure 5 after NAD+ addition (n = 4).

  
As expected the initial enhancement rate of fluorescence intensity reached
almost saturation as soon as using NAD+ concentrations higher then 250 nM
which matched the employed molecular beacon concentration. In this case the
molecular beacon, the second substrate for LigA, ought to be the limiting
factor for the enzymatic reaction, but not the NAD+ which is supposed to be
detected. Anyway, there seems to exist a linear dependence for the NAD+
concentration and the initial enhancement rate of fluorescence for NAD+
concentrations below the molecular beacon concentration. This is shown in
figure 7 in which the linear dependence is indicated by linear regression of
the data. As you can see the NAD+ concentration correlates with the average of
fluorescence enhancement rate in 200 s after NAD+ addition (initial velocity)
in a linear way as long as the NAD+ concentration was kept below the systems
capacity. This means that a specific fluorescence enhancemant rate is
characteristic for a particular NAD+ concentration caused by a fixed affinity
of LigA and NAD+ as a limiting substrate for DNA ligation. Referring to this,
the minimal detected NAD+ concentration was 2 nM.

  

[![](/wiki/images/thumb/d/d9/Bielefeld-
Germany-2011_NAD%2B_calibrationcurve.png/650px-Bielefeld-
Germany-2011_NAD%2B_calibrationcurve.png)](/File:Bielefeld-
Germany-2011_NAD%2B_calibrationcurve.png)

[](/File:Bielefeld-Germany-2011_NAD%2B_calibrationcurve.png "Enlarge")

**Figure 7: Initial velocity calibration curve for NAD +.** The initial
velocity, calculated from the average fluorescence enhancement rate in 200 s
after NAD+ addition as shown in figure 5, is plotted versus the employed NAD+
concentration. Data is fitted with linear regression for NAD+ concentrations
ranging from 0 to 200 nM (R² = 0.991, n = 4).

  
The fluorescence intensity enhancement is supposed to be caused by LigA
ligating the split target in a NAD+-dependent manner. This becomes clearly
first due to the absence of fluorescence intensity increase for the negative
control (0 nM NAD+) and second due to the lengthy fluorescence enhancement
after NAD+ addition suggesting enzymatic activity by LigA. This could be
directly visualized by taking pictures of the reactions endpoint. As
illustrated in figure 8 the fluorescence intensity for added LigA and NAD+
reached nearly the same level as for the positive control (complete target).
This was not the case when NAD+ was missing in the reaction mix with LigA and
the split target.

  

[![](/wiki/images/thumb/b/b1/Bielefeld-
Germany-2011_NAD%2B_bioassay_imaging.png/500px-Bielefeld-
Germany-2011_NAD%2B_bioassay_imaging.png)](/File:Bielefeld-
Germany-2011_NAD%2B_bioassay_imaging.png)

[](/File:Bielefeld-Germany-2011_NAD%2B_bioassay_imaging.png "Enlarge")

**Figure 8: Imaging the NAD +-dependence of LigA.** The fluorescence
intensities of 500 nM molecular beacon alone, added with 600 nM complementary
target, 600 nM complementary split target or 5 ng/µL LigA and NAD+ show that
LigA ligates the split target only in presence of NAD+ resulting in a
fluorescence intensity comparable to the one for the complete target. Image
was generated with a laser scanner and colored subsequently.

  

### Selectivity test

The specifity of LigA for its substrate NAD+ is important in order to couple
the NAD+ detection with investigated processes including NADH-dependent enzyme
reactions. The verification of the selectivity is an important matter to
exclude any unspecific reactions which might result in a NAD+-independent
fluorescence enhancement. Hence, a selectivity test for LigA was performed
with the analytes NADH, NADP+, NADPH, 3-ADAP (NAD+ with an exchanged
functional group at the nicotinamide ring system), ATP and ADP, and the
relative fluorescence enhancement rates in a NAD+ bioassay were compared with
the one for NAD+ (figure 9).

[![](/wiki/images/thumb/2/23/Bielefeld-Germany-2011_Selectivity-of-
LigA.png/650px-Bielefeld-Germany-2011_Selectivity-of-
LigA.png)](/File:Bielefeld-Germany-2011_Selectivity-of-LigA.png)

[](/File:Bielefeld-Germany-2011_Selectivity-of-LigA.png "Enlarge")

**Figure 9: LigA shows high selectivity for NAD +.** The final concentration
of all analytes was 100 nM. The responses were evaluated on the basis of the
average fluorescence enhancement rate in a range of 200 s after addition of
each analyte into the NAD+ bioassay. The dotted line marks the threshold
indicating the intensity of background signal. All data are normalized to the
NAD+ value (n = 2).

  
The negative control (H2O) shows that there was a small background signal
about 5 % of the signal that was produced by NAD+ when using 100 nM of
analytes. This marks the threshold for fluorescence enhancement which is
caused by the employed analyte. NADH, NADPH and ATP were similar to the
negative control and can thereby seen as analytes that do not enable DNA
ligation by LigA. Only the values for the three analytes NADP+, 3-APAD and ADP
were above the threshold, but they were constantly below 10 % of the NAD+
signal. This leads to the suggestion that LigA ist highly selective for NAD+
even in presence of structurally similar analytes. This makes the NAD+
bioassay and the associated enzyme LigA suitable for investigating NADH-
dependent enzyme reactions or measuring NAD+ in biological analyte mixtures
such as cell lysates.

  

### Coupled enzyme reaction

The coupling of the LigA-including NAD+ detection system was performed with a
NADH-dependent enzymatic reaction which was in concrete the conversion of
pyruvate to L-lactate by lactic acid dehydrogenase (LDH) from _E. coli_. For
this the LDH reaction was done with an excess of NADH and various pyruvate
concentrations. Afterwards, the reaction mix was transferred to a NAD+
bioassay and the fluorescence intensity was monitored. In figure 10 the
normalized initial fluorescence enhancement rates for the employed pyruvate
concentrations are indicated. The calculated initial velocity was then plotted
against the pyruvate concentration (figure 11).

[![](/wiki/images/thumb/4/4e/Bielefeld-Germany-2011_Fluorescence-enhancement-
after_addition-of-LDH-reaction-mix.png/650px-Bielefeld-
Germany-2011_Fluorescence-enhancement-after_addition-of-LDH-reaction-
mix.png)](/File:Bielefeld-Germany-2011_Fluorescence-enhancement-
after_addition-of-LDH-reaction-mix.png)

[](/File:Bielefeld-Germany-2011_Fluorescence-enhancement-after_addition-of-
LDH-reaction-mix.png "Enlarge")

**Figure 10: Fluorescence enhancement rate after addition of LDH reaction mix
with various pyruvate concentrations into a LigA-dependent NAD + bioassay.**
50 ng/µL LDH was preincubated with 100 µM NADH and different pyruvate
concentrations for 2 min at 37 °C. 1 µL of the LDH reaction mix was then added
to a NAD+ bioassay, composed of a nicked DNA substrate (250 nM molecular
beacon hybridized to a split target) and 6.5 ng/µL LigA, and the fluorescence
intensity was monitored. For illustration the fluorescence intensity is
normalized for each measurement series to the first measured fluorescence
value after LDH reaction mix addition (n = 4).

[![](/wiki/images/thumb/5/50/Bielefeld-Germany-2011_Pyruvate-calibration-
curve.png/650px-Bielefeld-Germany-2011_Pyruvate-calibration-
curve.png)](/File:Bielefeld-Germany-2011_Pyruvate-calibration-curve.png)

[](/File:Bielefeld-Germany-2011_Pyruvate-calibration-curve.png "Enlarge")

**Figure 11: Initial velocity calibration curve for pyruvate.** The initial
velocity, calculated from the average fluorescence enhancement rate in 100 s
after LDH reaction mix addition into a LigA-dependent NAD+ bioassay, is
plotted versus the employed pyruvate concentration. Data is fitted with linear
regression for pyruvate concentrations ranging from 0 to 10 µM (R² = 0.951, n
= 4).

  
The addition of the LDH reaction mix resulted in a characteristic fluorescence
enhancement rate depending on the employed pyruvate concentration. The
existing correlation between both parameters seemed to be a linear. That the
signal for 0 µM pyruvate was remarkably high could be the result of an
pyruvate-independent transfer of electrons from NADH to the active site
histidine of LDH under formation of NAD+. The limit of detection for pyruvate
seemed to be near 1 µM pyruvate. That this value was not in nano molarity
scale is firstly caused by 100-fold dilution of the LDH reaction mix after
addition to the NAD+ bioassay and secondly by the fact that LDH does not
necessarily converte 100 % of the pyruvate to L-lactate. Nevertheless, for the
LigA based NAD+ detection system it has been proven that it can be coupled to
NADH-dependent enzymatic reactions. This makes the NAD+ bioassay suitable for
a wide range of biological studies dealing with the ubiquitous cofactors
NADH/NAD+.

  

### Molecular cloning

Because DNA ligase from _E. coli_ is commercially aquirable for cloning
purposes LigA
([BBa_K525710](http://parts.igem.org/wiki/index.php/Part:BBa_K525710)) was
tested whether it is also suitable for molecular cloning procedures. Hence,
cloning of _rfp_ into a pSB1C3 vector was performed with LigA. Ligation was
done with NAD+ bioassay buffer containing 10 mM NAD+ at 37 °C or 22 °C and the
vectors were transformed into _E. coli_ KRX by electroporation. After growth
over night at 37 °C on petri dishes the results were documented (figure 12).

[![](/wiki/images/thumb/a/a9/Bielefeld-Germany-2011_LigAmolecularcloning-
control.JPG/700px-Bielefeld-Germany-2011_LigAmolecularcloning-
control.JPG.png)](/File:Bielefeld-Germany-2011_LigAmolecularcloning-
control.JPG)

[](/File:Bielefeld-Germany-2011_LigAmolecularcloning-control.JPG "Enlarge")

**Figure 12: _rfp_ cloning without any ligase (A) or with ligation by DNA
ligase from _E. coli_ (LigA) at 37 °C (B) and 22 °C (C).** The picture shows
the grown colonies after transformation of a control sample ( _rfp_ and cut
pSB1C3) or ligation sample ( _rfp_ and cut pSB1C3 with LigA) into _E. coli_
KRX.

By analyzing the number of colony forming units and color of the colonies the
ligation efficiency of LigA should be assessable. As shown for the sample
without any ligase there were only a few clones able to grow on
chloramphenicol supplemented LB medium. A much bigger number of clones was
observed when ligation by LigA was performed at 22 °C. Furthermore, there were
a lot of red colonies indicating positive clones. Deductively, LigA might be
useful for (large-scale) molecular cloning procedures which could be
beneficial because of its easy production in _E. coli_ and simple
purification.  

  

### Conclusions

LigA, the NAD+-dependent DNA ligase from _E. coli_ , can be utilized for a
molecular beacon based bioassay detecting NAD+ in nano molarity scale (limit
of detection: 2 nM). It has been shown that the NAD+ concentration determines
the initial fluorescence enhancement rate as a result of ligation of the
nicked DNA substrate (250 nM molecular beacon hybridized to a split target).
There is a linear dependence for both parameters while the NAD+ concentration
is kept below the molecular beacon concentration. By varying the molecular
beacon concentration the range for linear dependence of NAD+ concentration and
initial velocity of LigA might be expandable. Furthermore, LigA shows a high
selectivity for NAD+ which means that DNA ligation upon NADH consumption does
not occur in a measurable manner. This enables NAD+ detection even in presence
of structurally similar analytes like NADH, NADP+, NADPH, 3-ADAP, ATP or ADP.
By meeting this requirements the NAD+ bioassay is suitable for investigations
of coupled NADH-dependent enzyme reactions and monitoring of NAD+ pools in
cell lysates. Finally, LigA seems to be suitable as a model system for
bacterial DNA ligases due to the highly conserved functional domains
throughout the bacterial kingdom. This could be useful in terms of antibiotic
drug design utilizing the bacterial DNA ligases specifity for NAD+ as a
cofactor.  

  

### Methods

#### Characterization of molecular beacons

Molecular beacons are nucleic acid probes that fluoresce upon hybridization to
a complementary target. To check whether they are suitable for a NAD+ bioassay
involving LigA they can be characterized by thermal profile analysis and
determination of the signal-to-background ratio. Detailed background
information, tools for the correct design of molecular beacons, a list of
synthesis companies as well as protocols for the synthesis and
characterization of molecular beacons can be found
[here](http://www.molecular-beacons.com/default.htm).  
The used nucleic acids were designed as follows:

Name  | Sequence  | Label  
---|---|---  
Molecular beacon  | 5'-CCT CTC CGT GTC TTG TAC TTC CCG TCA GAG AGG-3'  |
5'-FAM, 3'-Dabcyl  
Target  | 5'-GAC GGG AAG TAC AAG ACA C-3'  |  
Split target  | 5'-TAC AAG ACA C-3'  | 5'-phosphate  
5'-GAC GGG AAG-3'  |  
  
  
The molecular beacon was purchased from
[Eurogentec](http://www.eurogentec.com/eu-home.html) and labeled with 6-FAM
(fluorophor) and Dabcyl (appropriate quencher for 6-FAM). For 6-FAM labeled
molecular beacon the optimal excitation wavelength was 490 nm and the emission
wavelength was 530 nm. All measurements were done in a spectrofluorometer and
in NAD+ bioassay buffer (see chapter NAD+ bioassay) at 37 °C. The calculation
of the signal-to-background ratio (S/B) is as follows:

[![Bielefeld-Germany-2011 MB-signaltobackgroundratio-
calculation.png](/wiki/images/thumb/0/0a/Bielefeld-Germany-2011_MB-
signaltobackgroundratio-calculation.png/450px-Bielefeld-Germany-2011_MB-
signaltobackgroundratio-calculation.png)](/File:Bielefeld-Germany-2011_MB-
signaltobackgroundratio-calculation.png)

#### Purification of LigA

The purification of the overexpressed LigA in _E. coli_ was performed under
native conditions and Ni-NTA columns were used utilizing the recombinant
proteins C-terminal 6xHis-tag.

  * Prepare an overnight culture of [_E. coli_ KRX](http://www.promega.com/products/cloning-and-dna-markers/cloning-tools-and-competent-cells/bacterial-strains-and-competent-cells/single-step-_krx_-competent-cells/) carrying the plasmid with [BBa_K525710](http://parts.igem.org/wiki/index.php/Part:BBa_K525710) in 30 mL LB medium containing 20 µg mL-1 chloramphenicol at 37 °C.
  * Dilute the overnight culture in 100 mL autoinduction medium (LB medium supplemented with 0,1 % L-rhamnose and 20 µg mL-1 chloramphenicol) to an OD600=0.1 and harvest the cells after 4 h growth at 37 °C.
  * Use the centrifuged cells immediately for protein purification or store them at -20 °C.
  * Resuspend a cell culture pellet in 5 mL binding buffer (20 mM Na3PO4 pH 7.4, 500 mM NaCl, 20 mM imidazole) per gramm biomass.
  * Add 10 mM MgCl2, 1 mg/mL lysozyme as well as 5 Units benzonase nuclease per 1 mL cell culture volume and mix thoroughly. 
  * Incubate at 4 °C for 30 min.
  * Centrifuge the lysate for 30 min at 10000 g (4 °C). Collect the supernatant (cleared lysate).
  * Prepare a [HisTrap™ FF crude column](http://www.gelifesciences.com/aptrix/upp01077.nsf/Content/Products?OpenDocument&moduleid=165904) ([GE Healthcare](http://www.gelifesciences.com/aptrix/upp01077.nsf/Content/life-sciences_homepage)) by equilibration with 5 column volumes of binding buffer. The following steps have to be performed at 4 °C.
  * Load the cleared lysate onto the HisTrap™ FF crude column and collect flow-through.
  * Wash the HisTrap™ FF crude column with 10 column volumes of binding buffer and collect flow-through.
  * Elute the protein successively with 5 mL of binding buffer that contains different concentrations of imidazole (e.g. 40, 60, 100, 300 and 500 mM imidazole) and collect flow-through in 1 mL fractions.
  * For centrifugal ultrafiltration use [Vivaspin 500](http://www.sartorius-stedim.com/DE/en/Centrifugal-Ultrafiltration--Vivaspin-%26-Centrisart/Vivaspin-500/VS0111-VIVASPIN-500-5%2C000-MCWO-PES-25-BOX/6htcki6uex7/m8qc5rabaox/n8uj9p1cyz4/article.htm) ([Sartorius](http://www.sartorius.de/index.php?id=156&no_cache=1)) concentrators with a 5000 molecular weight cut-off PES membrane. Repeat the procedure several times by adding DNA ligase buffer (20 mM Tris-HCl pH 7.5, 50 mM NaCl, 20 % [v/v] Glycerol) for buffer exchange (remove of imidazole).
  * Check each purification step and especially the purity of the protein in its final condition by SDS-PAGE analysis.
  * Determine the protein concentration with a Bradford Protein Assay.
  * For long-time storage keep the protein at -20 °C.

  

#### Deadenylation

The majority of purified DNA ligase from _E. coli_ KRX is usually in the
adenylated form. But only the apoenzyme is really useful for the NAD+ bioassay
so that the AMP moiety has to be removed.

  * Mix purified LigA with deadenylation buffer (20 mM Tris-HCl pH 7.5, 50 mM NaCl, 4 mM MgCl2, 1 mM EDTA, 1 mM DTT, 10 mM NMN) and incubate for 20 min at 37 °C.
  * For centrifugal ultrafiltration use [Vivaspin 500](http://www.sartorius-stedim.com/DE/en/Centrifugal-Ultrafiltration--Vivaspin-%26-Centrisart/Vivaspin-500/VS0111-VIVASPIN-500-5%2C000-MCWO-PES-25-BOX/6htcki6uex7/m8qc5rabaox/n8uj9p1cyz4/article.htm) ([Sartorius](http://www.sartorius.de/index.php?id=156&no_cache=1)) concentrators with a 5000 molecular weight cut-off PES membrane. Repeat the procedure several times by adding DNA ligase buffer (20 mM Tris-HCl pH 7.5, 50 mM NaCl, 20 % [v/v] Glycerol) for buffer exchange (remove of NMN and potentially formed NAD+).
  * Determine the protein concentration with a Bradford Protein Assay.
  * For long-time storage keep the protein at -20 °C.

  

#### NAD+ bioassay

  * Prepare a master mix composed of 250 nM molecular beacon and 250 nM split target (notice that the 5'-end at the nick site has to be phosphorylated) in NAD+ bioassay buffer (50 mM Tris-HCl pH 8.0, 10 mM MgCl2, 2.5 mM CaCl2, 5 mM DTT, 0.05 % BSA).
  * Incubate for 20 min at 37 °C.
  * Add 500 ng purified and deadenylated LigA (final concentration 5 ng/µL).
  * Monitor the fluorescence with a spectrofluorometer at 37 °C and wait until it reaches equilibrium.
  * Add 1 µL NAD+ in different final concentrations (e.g. 0, 2, 5, 10, 20, 30, 50, 80, 100, 200, 300, 500, 1000, 10000 nM) filling the reaction mix up to 100 µL, stirr shortly and monitor the fluorescence at 37 °C again.
  * Determine the initial enhancement rate of fluorescence intensity for each NAD+ concentration (in a range of 200 s after NAD+ addition).
  * For the calibration curve plot the initial velocity versus the NAD+ concentration.

#### Imaging

  * It is a similar procedure to the one mentioned above except that you prepare 500 nM molecular beacon and 600 nM target or split target dissolved in bioassay buffer in a PCR tube. 
  * After incubation at 37 °C add 1000 ng LigA and possibly NAD+ in different concentrations filling the reaction mix up to 200 µL.
  * Visualize the fluorescence in a laser scanner or UV transilluminator and document the results.

  

#### Selectivity test

  * Follow the instructions for the NAD+ bioassay but use different analytes like NADH, NADP+, NADPH, [3-APAD](http://www.sigmaaldrich.com/catalog/ProductDetail.do?D7=0&N5=SEARCH_CONCAT_PNO%7CBRAND_KEY&N4=A5251%7CSIGMA&N25=0&QS=ON&F=SPEC), ATP, ADP or AMP in a final concentration of 100 nM.

  

#### Coupled enzyme reaction

  * To check whether the NAD+ bioassay is suitable for the analysis of NADH-dependent enzymatic reactions it was coupled with the NADH-dependent conversion of pyruvate to L-lactic acid by [lactic acid dehydrogenase (LDH)](http://www.sigmaaldrich.com/catalog/ProductDetail.do?lang=de&N4=59747%7CSIGMA&N5=SEARCH_CONCAT_PNO%7CBRAND_KEY&F=SPEC) from _E. coli_.
  * Prepare a master mix composed of 250 nM molecular beacon and 250 nM split target (notice that the 5'-end at the nick site has to be phosphorylated) in NAD+ bioassay buffer (50 mM Tris-HCl pH 8.0, 10 mM MgCl2, 2.5 mM CaCl2, 5 mM DTT, 0.05 % BSA).
  * Incubate for 20 min at 37 °C. 
  * Meanwhile, prepare a separate reaction mix composed of LDH reaction buffer (20 mM Tris-HCl pH 7.5, 50 mM NaCl), 100 µM NADH and 50 ng/µL LDH.
  * Incubate for 5 min at 37 °C.
  * Add 650 ng purified and deadenylated LigA to the molecular beacon reaction mix (final concentration 6.5 ng/µL).
  * Monitor the fluorescence with a spectrofluorometer at 37 °C and wait until it reaches equilibrium.
  * Add 1 µL pyruvate in different final concentrations (e.g. 0, 1, 2.5, 5, 7.5, 10 µM) to the LDH reaction mix.
  * Incubate for 2 min at 37 °C.
  * Add 1 µL LDH reaction mix to the equilibrated NAD+ bioassay solution filling the reaction mix up to 100 µL, stirr shortly and monitor the fluorescence at 37 °C again.
  * Determine the initial enhancement rate of fluorescence intensity for each pyruvate concentration (in a range of 100 s after LDH reaction mix addition).
  * For the calibration curve plot the initial velocity versus the pyruvate concentration.

  

### References

Gajiwala KS, Pinko C (2004) Structural Rearrangement Accompanying NAD+
Synthesis within a Bacterial DNA Ligase Crystal, [_Structure_
12(8):1449-1459](http://www.cell.com/structure/abstract/S0969-2126%2804%2900235-7).

Miesel L, Kravec C, Xin AT, McMonagle P, Ma S, Pichardo J, Feld B, Barrabee E,
Palermo R (2007) High-throughput assay for the adenylation reaction of
bacterial DNA ligase, [_Analytical Biochemistry_ 366:9-17](http://medical-
journals.healia.com/doc/17493575/A-high-throughput-assay-for-the-adenylation-
reaction-of-bacterial-DNA-ligase).

Nandakumar J, Nair PA, Shuman S (2007) Last Stop on the Road to Repair:
Structure of _E. coli_ DNA Ligase Bound to Nicked DNA-Adenylate, [_Molecular
Cell_ 26(2):257-271](http://www.cell.com/molecular-
cell/abstract/S1097-2765%2807%2900144-X).

Tang Z, Liu P, Ma C, Yang X, Wang K, Tan W, Lv X (2011) Molecular Beacon Based
Bioassay for Highly Sensitive and Selective Detection of Nicotinamide Adenine
Dinucleotide and the Activity of Alanine Aminotransferase, [_Anal Chem_
83(7):2505-2510](http://pubs.acs.org/doi/abs/10.1021/ac102742k).

Tyagi S, Kramer FR (1996) Molecular beacons: probes that fluoresce upon
hybridization, [_Nature Biotechnology_
14:303-308](http://www.nature.com/nbt/journal/v14/n3/abs/nbt0396-303.html).

  

  

## Functional Parameters: Austin_UTexas

BBa_K525710 parameters

### Burden Imposed by this Part:

![](http://parts.igem.org/wiki/images/f/fa/T--Austin_Utexas--
no_burden_icon.png)

**Burden Value: -2.2 ± 3.5%**

Burden is the percent reduction in the growth rate of _E. coli_ cells
transformed with a plasmid containing this BioBrick (± values are 95%
confidence limits). This BioBrick did not exhibit a burden that was
significantly greater than zero (i.e., it appears to have little to no impact
on growth). Therefore, users can depend on this part to remain stable for many
bacterial cell divisions and in large culture volumes. Refer to any one of the
[BBa_K3174002](http://parts.igem.org/Part:BBa_K3174002) \-
[BBa_K3174007](http://parts.igem.org/Part:BBa_K3174007) pages for more
information on the methods, an explanation of the sources of burden, and other
conclusions from a large-scale measurement project conducted by the [2019
Austin_UTexas team](http://2019.igem.org/Team:Austin_UTexas).

This functional parameter was added by the [2020 Austin_UTexas
team.](http://2020.igem.org/Team:Austin_UTexas/Contribution)

[[edit](http://parts.igem.org/partsdb/part_info.cgi?part_name=BBa_K525710)]

Categories

//cds/enzyme/ligase  
//chassis/prokaryote/ecoli  
//proteindomain/affinity  
//proteindomain/tail  
//rnap/bacteriophage/t7

Parameters

chassis| E. coli| control| PT7| origin| Escherichia coli| tag| His-tag

