---
layout: post
title:  "BBa_K523000"
author: Reporter
categories: [collection 1 - 2021,] 
image: assets/images/6.jpg
---


  * [main page](http://parts.igem.org/Part:BBa K523000)
  * [design](http://parts.igem.org/Part:BBa K523000:Design)
  * [experience](http://parts.igem.org/Part:BBa K523000:Experience)
  * [information](http://parts.igem.org/cgi/partsdb/part_info.cgi?part_name=BBa K523000)
  *   * 

[Released HQ 2013](http://parts.igem.org/Help:Part_Status_Box)

[Sample In stock](http://parts.igem.org/Help:Part_Status_Box)

Experience: Works

Not Used

[ Get This Part](http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K523000)

![](http://parts.igem.org/images/partbypart/icon_reporter.png)

Reporter

# Part:BBa_K523000

Designed by: Chris French, Allan Crossman, Sylvia Ispasanie   Group:
iGEM11_Edinburgh   (2011-06-28)

[![](/wiki/images/6/69/Edinburgh-INP-YFP-cells.jpg)](/File:Edinburgh-INP-YFP-
cells.jpg)

[](/File:Edinburgh-INP-YFP-cells.jpg "Enlarge")

K523000 driving expression of
[BBa_K523013](http://parts.igem.org/wiki/index.php/Part:BBa_K523013), which
includes a Yellow Fluorescent Protein sequence.

[![](/wiki/images/thumb/9/92/K523000-BlueWhiteSelection.jpg/167px-K523000-BlueWhiteSelection.jpg)](/File:K523000-BlueWhiteSelection.jpg)

[](/File:K523000-BlueWhiteSelection.jpg "Enlarge")

K523000 being used to prepare
[BBa_K523002](http://parts.igem.org/wiki/index.php/Part:BBa_K523002). White
colonies were selected.

Plac + lacZ; with BglII site

This is based on part
[BBa_J33207](http://parts.igem.org/wiki/index.php/Part:BBa_J33207). Like that
part, it contains the lac promoter and codes for 76 N-terminal residues of
LacZ (β-galactosidase), a peptide that complements the lacZ-Δ-M15 mutation in
common lab strains of _E. coli_. If these strains are grown on Xgal and IPTG,
colonies with this part will be blue.

The part can be used as a promoter or as a cloning vector.

  
**Usage as a promoter**

The part can be used simply as a (detectable) promoter. If added upstream of
an RBS and coding sequence, a poly-cistronic transcript will be generated with
LacZ followed by the other gene.

  
**Usage as a cloning vector**

The first four bases of the sequence add a BglII site (BglII sites have 6
bases, but the first 2 are provided by the standard BioBrick prefix). This
allows K523000 to be used as a cloning vector for creating new BioBricks in
the following way: the forward primer for the new BioBrick should have a BglII
site added, and the reverse primer should have a SpeI site added.

The PCR product and the plasmid containing this part can then both be digested
with BglII and SpeI. After purification, mixing, ligation, and transformation,
colonies with plasmids that have successfully gained the new BioBrick will be
white, while those still containing the lacZ part will be blue (on Xgal+IPTG
plates).

One might ask: why not use the standard flanking BioBrick sites XbaI and SpeI
for this procedure? The answer is that these produce compatible sticky ends
and so the new part would circularise. Thus, the normal method is to use one
very long primer so the part can be inserted via EcoRI/SpeI or XbaI/PstI. But
our method avoids the requirement for such a long primer.

  
**Procedure**

To use as a vector for cloning a new part, first PCR the source gene so as to
add a BglII site ( **agatct** ) upstream of the gene, and a "t" followed by a
SpeI site downstream of it (so: **t actagt**).

The gene must not contain any BglII sites or SpeI sites. However, genes
containing EcoRI, XbaI, or PstI sites can be successfully cloned this way;
afterwards site-specific mutagenesis must be carried out to remove them, so as
to produce an RFC10-compatible BioBrick.

Primers for an imaginary gene might look like this:

  * Forward: aaa **agatct** atg tat tcg gct aat gat
  * Reverse: aaa **actagt a** tta tta gct ttc gtg gca gat cca

(The "aaa" bases are just to give the restriction enzymes room to work.)

After PCR, purify the PCR product and then follow this procedure... (or
something similar. This is Edinburgh 2011's exact procedure. For detailed
protocols, see the [Open Wetware: French
Lab](http://openwetware.org/wiki/French_Lab) page.)

[![K523000
diagram.png](/wiki/images/a/a6/K523000_diagram.png)](/File:K523000_diagram.png)

Replate some white colonies, then miniprep them and carry out verification
sequencing. You should now have an RFC10-compatible BioBrick with a full
prefix and suffix. The first four bases of the part will be **"atct"**. For an
example of a BioBrick produced in this way, see
[BBa_K523002](http://parts.igem.org/wiki/index.php/Part:BBa_K523002).

Parts produced in this way will also be compatible with Edinburgh's 2011
BioSandwich protocol (BBF RFC 81), though its use is entirely optional.

  
**Sequence and Features**

  

Assembly Compatibility:

  * 10

COMPATIBLE WITH RFC[10]

  * 12

COMPATIBLE WITH RFC[12]

  * 21

COMPATIBLE WITH RFC[21]

  * 23

COMPATIBLE WITH RFC[23]

  * 25

COMPATIBLE WITH RFC[25]

  * 1000

COMPATIBLE WITH RFC[1000]

  

  

  

## Characterization : YAU-China 2019

YAU-China team verified the function of this part in E.coli and got the
following results:  

[![The transformation results of laboratory sequence PA4110 related to this
part.](/wiki/images/thumb/a/a9/T--YAU-China--iGEM2019-BBa_-5.png/600px-T--YAU-
China--iGEM2019-BBa_-5.png)](/File:T--YAU-China--iGEM2019-BBa_-5.png)

[](/File:T--YAU-China--iGEM2019-BBa_-5.png "Enlarge")

Figure 1. The transformation results of laboratory sequence PA4110 related to
this part

[![PCR agarose gel electrophoresis: 1, 2, 3 were white colonies, 4, 5 and 6
were blue colonies.](/wiki/images/9/9f/T--YAU-China--
iGEM2019-BBa_-7.png)](/File:T--YAU-China--iGEM2019-BBa_-7.png)

[](/File:T--YAU-China--iGEM2019-BBa_-7.png "Enlarge")

Figure 2. PCR agarose gel electrophoresis: 1, 2, 3 were white colonies, 4, 5
and 6 were blue colonies.

Then we tested the gene connection efficiency of the plasmid and the
proportion of its positive colonies.

This is our transformation diagram of four groups of plasmids.

[![](/wiki/images/thumb/7/7c/T--YAU-China--iGEM2019-BBa_-19.jpg/600px-T--YAU-
China--iGEM2019-BBa_-19.jpg)](/File:T--YAU-China--iGEM2019-BBa_-19.jpg)

[](/File:T--YAU-China--iGEM2019-BBa_-19.jpg "Enlarge")

Figure 3.

[![Gene linkage data of BBa_K523000.](/wiki/images/thumb/c/cd/T--YAU-China--
iGEM2019-BBa_-16.jpg/600px-T--YAU-China--iGEM2019-BBa_-16.jpg.png)](/File:T--
YAU-China--iGEM2019-BBa_-16.jpg)

[](/File:T--YAU-China--iGEM2019-BBa_-16.jpg "Enlarge")

Figure 4. Gene linkage data of BBa_K523000.

[![Statistical chart of correct number of positive
clones.](/wiki/images/thumb/2/22/T--YAU-China--iGEM2019-BBa_-17.png/600px-T--
YAU-China--iGEM2019-BBa_-17.png)](/File:T--YAU-China--iGEM2019-BBa_-17.png)

[](/File:T--YAU-China--iGEM2019-BBa_-17.png "Enlarge")

Figure 5. Statistical chart of correct number of positive clones.

Results: the correct rate of positive cloning was 97.5%.

  

  

## Functional Parameters: Austin_UTexas

BBa_K523000 parameters

### Burden Imposed by this Part:

![](http://parts.igem.org/wiki/images/f/fa/T--Austin_Utexas--
no_burden_icon.png)

**Burden Value: 5.7 ± 6.1%**

Burden is the percent reduction in the growth rate of _E. coli_ cells
transformed with a plasmid containing this BioBrick (± values are 95%
confidence limits). This BioBrick did not exhibit a burden that was
significantly greater than zero (i.e., it appears to have little to no impact
on growth). Therefore, users can depend on this part to remain stable for many
bacterial cell divisions and in large culture volumes. Refer to any one of the
[BBa_K3174002](http://parts.igem.org/Part:BBa_K3174002) \-
[BBa_K3174007](http://parts.igem.org/Part:BBa_K3174007) pages for more
information on the methods, an explanation of the sources of burden, and other
conclusions from a large-scale measurement project conducted by the [2019
Austin_UTexas team](http://2019.igem.org/Team:Austin_UTexas).

This functional parameter was added by the [2020 Austin_UTexas
team.](http://2020.igem.org/Team:Austin_UTexas/Contribution)

[[edit](http://parts.igem.org/partsdb/part_info.cgi?part_name=BBa_K523000)]

Categories

Parameters

None|

