---
layout: post
title:  "BBa_K598000"
author: Reporter
categories: [collection 1 - 2021,chassis, prokaryote, ecoli, direction, forward, plasmidbackbone, copynumber, high, regulation, negative] 
image: assets/images/6.jpg
---


  * [main page](http://parts.igem.org/Part:BBa K598000)
  * [design](http://parts.igem.org/Part:BBa K598000:Design)
  * [experience](http://parts.igem.org/Part:BBa K598000:Experience)
  * [information](http://parts.igem.org/cgi/partsdb/part_info.cgi?part_name=BBa K598000)
  *   * 

[Released HQ 2013](http://parts.igem.org/Help:Part_Status_Box)

[Sample In stock](http://parts.igem.org/Help:Part_Status_Box)

Experience: Works

Not Used

[ Get This Part](http://parts.igem.org/partsdb/get_part.cgi?part=BBa_K598000)

![](http://parts.igem.org/images/partbypart/icon_reporter.png)

Reporter

# Part:BBa_K598000

Designed by: Yangyang ZHAO, Yan GONG, Zhenrun ZHANG, Qingyang XIAO   Group:
iGEM11_pku_riboclamp   (2011-09-18)

TPP Down-regulated Hammerhead Ribozyme 2.5 with Native RBS+E0040+B0015

This is a GFP generator regulated by thiamine pyrophosphate (TPP)-responsive
ribozyme 2.5. Upon adding TPP, the expression level of downsteam GFP would be
down-regulated, and the extent of down-regulation corresponds to different
concentrations of TPP.

  

Sequence and Features

  

Assembly Compatibility:

  * 10

COMPATIBLE WITH RFC[10]

  * 12

COMPATIBLE WITH RFC[12]

  * 21

INCOMPATIBLE WITH RFC[21]

Illegal BamHI site found at 121  

  * 23

COMPATIBLE WITH RFC[23]

  * 25

COMPATIBLE WITH RFC[25]

  * 1000

INCOMPATIBLE WITH RFC[1000]

Illegal BsaI.rc site found at 794  

  

## Background

Hammerhead ribozymes are small self-cleaving RNAs, first discovered in
satellite RNAs of plant viruses that catalyze a specific phosphodiester bond
isomerization reaction in the course of rolling-circle replication [1]. More
recently a full-length hammerhead ribozyme from _Schistosoma mansoni_ is being
more frequently utilized for application. As shown in **Figure 1A** , this
hammerhead ribozyme can be truncated to a minimal, catalytically active motif
consisting of three base-pairing stems (marked in colors) flanking a central
core of 15 mostly invariant nucleotides (marked in frame). And the conserved
central bases are essential for the hammerhead ribozyme’s catalytic activity
[1]. The tertiary structure shown in **Figure 1B** indicates that the
secondary structure of the _Schistosoma_ hammerhead ribozyme can be distorted
into a uridine turn because of distant loop/bulge interaction which induces
changes in stem II while simultaneously unwinding stem I. For the basic
catalytic function of hammerhead ribozyme, the active site for self-cleaving
of Schistosoma hammerhead ribozyme resides between stem III and stem I, as
shown in **Figure 1A**.

  
It has been reported previously that mRNAs encoding enzymes involved in
thiamine (vitamin B1) biosynthesis in Escherichia coli can bind thiamine or
its pyrophosphate derivative without the assistance from protein cofactors
[2]. These ligand-binding mRNAs actually possess thiamine or pyrophosphate
binding domain, called aptamer, in which the binding event can bring about a
conformational change which is important for genetic control. This natural
thiamine pyrophosphate (TPP) aptamer can bind to TPP specifically and a
defined structure is stabilized. As shown in **Figure 1C** , upon addition of
TPP, TPP can bind loop in green through non-covalent bond. **Figure 1D** shows
the tertiary structure of natural TPP aptamer binding to TPP.

  

[![](/wiki/images/thumb/7/7e/PekingR_A.jpg/600px-
PekingR_A.jpg)](/File:PekingR_A.jpg)

[](/File:PekingR_A.jpg "Enlarge")

**Figure 1** The schematic structures of _Schistosoma_ hammerhead ribozyme and
natural TPP aptamer. **A)** The secondary structure of _Schistosoma_
hammerhead ribozyme. Three base-pairing stems are shaded in colors. The part
in yellow represents stem III, which is later modified to be the linker
between hammerhead ribozyme and aptamer. The part in blue represents stem II,
and the purple and red ones stand for two parts of stem I. The sequence in
frame represents for the conserved nucleotides. The red arrow points to the
scissile bond. **B)** The tertiary structure of _Schistosoma_ hammerhead
ribozyme. The cyan part indicates the fragment of mRNA after cleavage. The red
one indicates the active site for self-cleavage. The linker between hammerhead
ribozyme and aptamer is shown in yellow. **C)** The secondary structure of
natural TPP aptamer. TPP can bind to loop in green through non-covalent bond
and the part marked in yellow indicates the linker between hammerhead ribozyme
and apatamer. **D)** The tertiary structure of natural TPP aptamer. The three-
dimensional segment in blue is TPP, and the yellow part represents the linker
between _Schistosoma_ hammerhead ribozyme and natural TPP aptamer. Nucleotides
that bind to TPP are shown in green.

## Original Design of TPP Ligand Responsive Hammerhead Ribozyme

In fact, the natural aptamer domain of the TPP riboswitch can be exploited to
construct very efficient ribozyme-based artificial switches that regulate gene
expression, demonstrated by Markus Wieland _et al._ [3]. To couple the natural
TPP aptamer riboswitch with Schistosoma hammerhead ribozyme, stem III of
_Schistosoma_ hammerhead ribozyme and yellow shaded stem of TPP aptamer in
**Figure 1C** were modified to construct linker between hammerhead ribozyme
and aptamer. The resulting artificial ribozymes functioned with high
performance, whose highest fold reached 1000.

Markus Wieland _et al._ have created several mutants of the constructed
ribozyme-based TPP-responsive artificial ribozyme switches (TPP ribozyme)[3].
We chose two of the mutants in our project, one of which can activate
downstream gene expression upon adding TPP, numbered 1.20, and the other would
inhibit downstream gene expression when TPP added, numbered 2.5. The secondary
structure of TPP ribozyme 2.5 is shown in **Figure 2**. The lower part of the
structure is natural TPP aptamer riboswitch, and the upper part is Schistosoma
hammerhead ribozyme. Stem III in green indicates the linker between aptamer
and hammerhead ribozyme, the pairing nucleotides of which is the only
distinction between TPP ribozyme 1.20 and 2.5.

  

![PekingR_Tpp.jpg](http://parts.igem.org/wiki/images/a/a8/PekingR_Tpp.jpg) |
![Peking-R_C.jpg](http://parts.igem.org/wiki/images/5/5f/Peking-R_C.jpg)  
---|---  
**Figure 2** Secondary structure of artificial thiamine pyrophosphate (TPP)
ribozymes 2.5. Natural TPP aptamer domain (blue) is fused to stem III of the
Schistosoma hammerhead ribozyme. The linker between aptamer and hammerhead
ribozyme is shown in green. Stems are indicated by roman numerals; rate-
enhancing interaction between stem I and stem II are shown as gray lines; the
cleavage site is marked by a red arrow. RBS is shaded in pink and the
translation start code (AUG) is shaded in black. The figure was modified from
[3]. | **Figure 3** Allosteric mechanism of TPP ribozymes. Upon self-cleavage
the RBS would be released from pairing, thus ribosome could get access to RBS
and initiate translation of the downstream gene . **Upper)** Mechanism for TPP
ribozyme 1.20. Upon addition of TPP, TPP ribozyme 1.20 adopts a conformation
that facilitates the self-cleavage of hammerhead domain. **Lower)** Mechanism
for TPP ribozyme 2.5. When TPP added, TPP ribozyme 2.5 would change to a
conformation that hinders the self-cleavage of hammerhead domain. Red: RBS
sequence, blue: natural TPP aptamer, green: linker between aptamer and
hammerhead domain, black: Schistosoma hammerhead ribozyme, red arrow: self-
cleavage site. The figure is modified from [3].  
  
  
The ribosomal binding site (RBS) of TPP ribozyme locates at the extended stem
(shaded red in **Figure 2** ). The _Schistosoma_ hammerhead domain in TPP
ribozyme could perform self-cleavage when posed in an appropriate
conformation, and upon self-cleavage the RBS would be released from pairing,
thus ribosome could get access to RBS and initiate translation of the
downstream gene. Though similar in secondary structure, TPP ribozyme 1.20 and
2.5 undergo different mechanisms to regulate the translation of downstream
gene. Upon addition of TPP, the aptamer domain would bind to TPP; while TPP
ribozyme 1.20 would change to a conformation that is suitable for hammerhead
domain to cleave itself, TPP ribozyme 2.5 would undergo a conformational
change that would decelerate the self-cleaving rate of hammerhead domain.
Therefore, upon adding TPP, TPP ribozyme 1.20 would facilitate the translation
of downstream gene, whereas TPP ribozyme 2.5 would decrease the translation
strength of downstream gene ( **Figure 3** ).

  

## Manipulating and Experimental Data

We designed several experiments to demonstrate that TPP ribozymes are truly
modular RNA controllers, independent of sequence context. Firstly we
constructed Part [BBa_K598000](http://parts.igem.org/Part:BBa_K598000)(
**Figure 4** ). This part consists of TPP ribozyme 2.5 kindly provided by
Prof. Hartig’s lab, [BBa_E0040](http://parts.igem.org/Part:BBa_E0040) and
[BBa_B0015](http://parts.igem.org/Part:BBa_B0015).

[![](/wiki/images/thumb/d/d1/PekingR_fp.png/600px-
PekingR_fp.png)](/File:PekingR_fp.png)

[](/File:PekingR_fp.png "Enlarge")

**Figure 4** Construction of TPP Down-regulated Hammerhead Ribozyme 2.5 with
Native RBS+E0040+B0015. This part consists of TPP ribozyme 2.5 with native RBS
(AAGGAGAT), [BBa_E0040](http://parts.igem.org/Part:BBa_E0040) and
[BBa_B0015](http://parts.igem.org/Part:BBa_B0015). It is obtained by PCR from
inactive TPP-HHAz 2.5 [3] which is kindly provided by Markus Wieland _et al._
, and then inserted into pSB1C3 through standard assembly.

Then we constructed three different constructs based on this part, along with
one construct kindly provided by Prof. Hartig’s lab ( **Figure 5** ). The
plasmid provided by Prof. Hartig’s lab contain TPP ribozyme 2.5 with an
upstream T7 promoter, and the downstream coding sequence is a GFP gene,
followed by a T7 terminator ( **Figure 5A** ). We termed this construct T7-2.5
([BBa_K598016](http://parts.igem.org/Part:BBa_K598016)).

To prove that the performance of TPP ribozymes is not relevant to the
downstream coding sequence, we inserted first 36 base pairs of the coding
sequence from Part BBa_E0040 ahead of the GFP coding sequence into the plasmid
( **Figure 5B** ). The construct was termed 36-2.5.

To further investigate whether the performance of TPP ribozymes is influenced
by upstream promoter and downsteam terminator, we inserted pBAD promoter
([BBa_I13453](http://parts.igem.org/Part:BBa_I13453)) into the upstream of
[BBa_K598000](http://parts.igem.org/Part:BBa_K598000) and constructed
[BBa_K598011](http://parts.igem.org/Part:BBa_K598011)( **Figure 5C** ). This
construct was named pBAD-2.5.

Another plasmid were constructed by inserting first 36 base pairs of CI gene
ahead of GFP coding sequence in pBAD-2.5 ( **Figure 5D** ). This was named
CI-2.5.

[![](/wiki/images/thumb/8/87/PekingR_ZYY9.png/600px-
PekingR_ZYY9.png)](/File:PekingR_ZYY9.png)

[](/File:PekingR_ZYY9.png "Enlarge")

**Figure 5** Scheme of constructs designed to demonstrate the modularity of
TPP ribozymes. **A)** T7-2.5
([BBa_K598016](http://parts.igem.org/Part:BBa_K598016))consists of T7
promoter, TPP ribozyme, GFP and T7 terminator with native RBS. **B)** 36-2.5
were constructed by inserting the first 36 base pairs of BBa_E0040 between TPP
ribozyme and GFP. **C)** pBAD-2.5
([BBa_K598011](http://parts.igem.org/Part:BBa_K598011))consists of pBAD
promoter ([BBa_I13453](http://parts.igem.org/Part:BBa_I13453)) , TPP ribozyme,
([BBa_E0040](http://parts.igem.org/Part:BBa_E0040)) and
([BBa_B0015](http://parts.igem.org/Part:BBa_B0015)) with native RBS
(AAGGAGAT). **D)** CI-1.20/CI-2.5 were constructed by adding the first 36 base
pairs of CI ahead of coding sequence of E0040.

After transforming these four constructs, namely T7-2.5, 36-2.5, pBAD-2.5, and
CI-2.5, into _E. coli_ DH5α cells respectively, the bacteria were
characterized in M9 medium with TPP concentration gradient. The result turned
out that corresponding working curves of the four constructs overlapped to a
large extent ( **Figure 6** ), despite of different downstream coding sequence
(comparing T7-2.5 to 36-2.5, or pBAD-2.5 to CI-2.5), or different upstream
promoter or downstream terminator (comparing T7-2.5 to pBAD-2.5). These
experiments conclusively demonstrated that the performance of TPP ribozyme 2.5
was independent of their coding sequence context, rendering it a truly modular
RNA device to regulate gene expression.

  

[![](/wiki/images/thumb/8/8e/PekingR_TPP2.5%281%29.png/600px-
PekingR_TPP2.5%281%29.png)](/File:PekingR_TPP2.5\(1\).png)

[](/File:PekingR_TPP2.5\(1\).png "Enlarge")

**Figure 6** Working curves of TPP ribozyme 2.5 in different constructs. The
inhibition ratio is fluorescence intensity under given TPP concentrations
compared to that of without TPP. Constructed plasmids were transformed into
_E. coli_ DH5α cells and characterized in M9 medium with a TPP concentration
gradient of 0.001, 0.003, 0.01, 0.03, 0.1, 0.3, 1, 3 uM. T7-2.5 and 36-2.5
were induced by 1mM IPTG. pBAD-2.5 and CI-2.5 were induced by 1mM arabinose.

  

## Applications

This year our team developed a platform for soft-coding of genetic circuits
aiming at making screening fast, affordable and more predictable. The platform
is composed of a RNA controller toolkit and an RBS calculator as illustrated
in our project. To demonstrate the versatility and validity of the platform,
we utilized the platform to improve performance of two modular genetic
devices, AND gate and bistable switch. TPP hammerhead ribozyme performs well
during characterization, thus is elected as an component of the toolkit to
optimizing these devices.

  
In order to optimize the performance of AND gate, we firstly determined the
optimal translation strength of the T7ptag gene using our RNA controller
toolkit. By placing TPP hammerhead ribozyme 2.5 upstream of the coding
sequence, we obtained an AND gate modulator whose T7ptag gene translation rate
varies in response to TPP concentration ( **Figure 7a** ). By optimizing the
strength of translation, we are able to make up for the leakage in
transcription and a translation rate that endows the AND gate with
satisfactory performance ( **Figure 7b** ).

[![](/wiki/images/thumb/3/34/PekingR_parts_AG_TPP2.50_1.png/900px-
PekingR_parts_AG_TPP2.50_1.png)](/File:PekingR_parts_AG_TPP2.50_1.png)

[](/File:PekingR_parts_AG_TPP2.50_1.png "Enlarge")

**Figure 7** Optimization of AND gate performance using RNA controller(TPP
ribozyme).(a) Output fluorescence of the AND gate device without addition of
TPP ligand(corresponding to a △G of -5.78kJ/mol). (b)Output fluorescence of
the AND gate device with addition of maximal concentration of TPP
ligand(1μM，corresponding to a △G of -3.38kJ/mol). Vertical and horizontal axes
indicate logarithm of the concentrations of arabinose and salicylate
respectively. Apparently, addition of TPP ligand(which attenuates translation
strength) improved the AND gate performance by decreasing the area of region
for “ON” state. The two output color plots are mapped to their corresponding
positions in the full phase diagram in Figure 2, showing that they display
fair agreement with modeling results(white and yellow rectangular
respectively).

TPP 2.5 is also introduced into the bistable switch part (BBa_K228003) that
inherited from Peking 2007 iGEM team.(For the full information of bistable
switch, please refer to
[BBa_K598002](http://parts.igem.org/wiki/index.php?title=Part:BBa_K598002)) A
bistable switch with TPP 2.5 modifying the translation rate of _cI434_ gene is
constructed. (BBa_K598024) ( **Figure 8** )

[![](/wiki/images/thumb/d/df/Peking_R_bistable_tpp.png/900px-
Peking_R_bistable_tpp.png)](/File:Peking_R_bistable_tpp.png)

[](/File:Peking_R_bistable_tpp.png "Enlarge")

**Figure 8** Construction of the bistable switch device carrying the RNA
controller (TPP ribozyme, shown as the tuning switch named TPP2.5 in the
figure).

We set two experiment groups for characterizing this part: one without
addition of TPP and another with TPP sufficient for full induction of the RNA
controller’s functions (self-cleavage of ribozyme). The experimental results
are shown in **Figure 9**. It can be seen that the group with excess TPP
(down-regulated translation strength of _cI434_ gene) indeed displayed
bistability. Thus, it has been indicated that TPP ligand responsive hammerhead
ribozyme will be suitable for modifying nonlinear and non-Boolean logic
genetic device like bistble switch.

[![](/wiki/images/thumb/d/d9/Peking_R_bistable_tpp_Results.png/900px-
Peking_R_bistable_tpp_Results.png)](/File:Peking_R_bistable_tpp_Results.png)

[](/File:Peking_R_bistable_tpp_Results.png "Enlarge")

**Figure 9** Fluorescence stereomicroscopic images of E.coli colonies with and
without TPP treatment.(A)E.coli colonies without TPP treatment(no decrease in
translation rate) are all green(high CI434/low CI state), displaying
monostability of the genetic device. (B)E.coli colonies with TPP treatment(no
decrease in translation rate) are a mixture of green(high CI434/low CI state)
and red(low CI434/high CI state) colonies, displaying bistability of the
genetic device. (C)Experimental results are mapped to the simulated “green”
proportion–△G curve.

We further applied this platform to optimize a segment of violacein
biosynthetic pathway, and achieved producing purer desired products. To see
more information, please go to
[BBa_K598019](http://parts.igem.org/wiki/index.php?title=Part:BBa_K598019).

[![](/wiki/images/thumb/0/0e/PekingR-Jerry1.png/800px-PekingR-
Jerry1.png)](/File:PekingR-Jerry1.png)

[](/File:PekingR-Jerry1.png "Enlarge")

**Figure 10** **(A)** Design of constructs to fine-tune violacein biosynthetic
pathway. Hexagon: Stem-loop terminator
([BBa_B0015](http://parts.igem.org/wiki/index.php?title=Part:BBa_B0015)); Bent
arrow: pBAD promoter
(([BBa_I13453](http://parts.igem.org/wiki/index.php?title=Part:BBa_I13453));
Oval: Ribosomal binding site; Straight arrow: Coding sequence originated from
Part:[BBa_K274003](http://parts.igem.org/wiki/index.php?title=Part:BBa_K274003);
ribbon shape: TPP-responsive ribozyme. **(B)** Decreasing the amount of VioE
may lower the pool of prodeoxyviolacein, hence lowering the production of
deoxychromoviridans, thus we inserted a TPP down-regulated hammerhead ribozyme
(TPP ribozyme 2.5) in front of vioE, and the metabolic flux would favor
proviolacein. _Dash arrow_ : down-regulated. **(C)** Increasing the amount of
VioD may increase the yield of proviolacein, thus we inserted a TPP up-
regulated hammerhead ribozyme (TPP ribozyme 1.20) upstream of vioD, and the
metabolic flux would favor proviolacein. _Dash arrow_ : down-regulated;
_Filled arrow_ : up-regulated.

[![](/wiki/images/thumb/3/3b/PekingR-Jerry2.png/600px-PekingR-
Jerry2.png)](/File:PekingR-Jerry2.png)

[](/File:PekingR-Jerry2.png "Enlarge")

**Figure 11** HPLC results of the bacterial extraction when different
concentrations of TPP were presented in the cultures. When induced with
arabinose, the bacterial extraction produced two peaks eluted at 0.7-1.0 min (
**(b)** comparing to **(a)** ), and we tentatively assumed that the two peaks
corresponded to proviolacein ( _left_ ) and deoxychromoviridans ( _right_ ).
Upon adding TPP into the cultures, the amount of deoxychromoviridans
decreased, and the ratio between proviolacein and deoxychromoviridans
increased upon increasing TPP concentration ( **(b)** to **(f)** ), until
deoxychromoviridans was not detectable **(f)**.

## Supporting Information

Additionally, to further confirm that the rise or drop of the working curves
in **Figure 6** on different TPP concentrations was indeed the contribution of
TPP ribozyme 2.5, we constructed a plasmid as control by substituting TPP
ribozyme 2.5 in pBAD-2.5 [BBa_K598011](http://parts.igem.org/Part:BBa_K598011)
to the native RBS (AAGGAGAT) of TPP ribozyme 2.5, termed "TPP-RBS" ( **Figure
12** ). Another construct termed "pBAD-1.20" was also constructed by
substituting TPP ribozyme 2.5 to TPP ribozyme 1.20 in pBAD-2.5 construct.
Similar characterization was performed, and the result showed that the
fluorescence intensity produced by TPP-RBS fluctuated, yet not significant
enough to show a trend to increase or decrease when TPP concentration went up,
compared with the obvious fluorescence intensity change produced by pBAD-2.5
or pBAD-1.20( **Figure 13** ). Therefore, we can reach the conclusion that TPP
ribozyme 2.5 functioned modularly to regulate downstream gene’s translation
strength upon different concentrations of TPP.

  

[![](/wiki/images/thumb/c/c0/PekingR_ZYY5.png/600px-
PekingR_ZYY5.png)](/File:PekingR_ZYY5.png)

[](/File:PekingR_ZYY5.png "Enlarge")

**Figure 12** Construction of TPP-RBS. TPP-RBS consists of pBAD promoter,
native RBS, BBa_E0040 and BBa_B0015.

  

[![](/wiki/images/thumb/e/e4/PekingR_TPP-RBS%281%29.png/600px-PekingR_TPP-
RBS%281%29.png)](/File:PekingR_TPP-RBS\(1\).png)

[](/File:PekingR_TPP-RBS\(1\).png "Enlarge")

**Figure 13** The fluorescence intensity of TPP-RBS, pBAD-1.20 and pBAD-2.5
under different TPP concentrations. Ordinate axis indicates the fluorescence
intensity normalized by cell density. Constructed plasmids were transformed
into E. coli DH5a cells and characterized in M9 medium with a TPP
concentration gradient of 0, 0.001, 0.003, 0.01, 0.03, 0.1, 0.3, 1, 3uM, with
induction by 1mM arabinose.

  

## References

[1] Monika Martick and William G.Scott. (2006). Tertiary Contacts Distant from
the Active Site Prime a Ribozyme for Catalysis. Cell 126, 309-320

[2] Wade Winkler, Ali Nahvi Ronald R. Breaker. (2002). Thiamine Derivatives
Bind Messenger RNAs Directly to Regulate Bacterial Gene Expression. Nature
419, 952-956

[3] Markus Wieland, Armin Benz, Benedikt Klauser, and Jörg S. Hartig. (2009).
Artificial Ribozyme Switches Containing Natural Riboswitch Aptamer Domains.
Angew. Chem. 121, 2753-2756

[[edit](http://parts.igem.org/partsdb/part_info.cgi?part_name=BBa_K598000)]

Categories

//chassis/prokaryote/ecoli  
//direction/forward  
//plasmidbackbone/copynumber/high  
//regulation/negative

Parameters

emission| 509nm| excitation| 470nm| ligands| Thiamine Pyrophosphate (TPP)|
n/a| TPP Down-regulated Hammerhead Ribozyme 2.5 with Native RBS+E0040+B0015|
resistance|  chloramphenicol

